---
title: "Lake_Cadagno_Plots"
output: html_document
---

```{r setup, include=FALSE}

setwd("~/GitHub/Lake_Cadagno_microbial_loop_Saini_et_al_2021/Fig.1-4_Suppl")

##Installing all required packages

set.seed(893689)
#automatic install packages
RequiredPackages <- c("ggplot2", "Rcpp", "vegan", "dplyr", "scales", "grid", "reshape2" ,"plyr","magrittr" ,"randomcoloR", "devtools", "remotes", "geosphere", "matrixStats" ,"data.table" ,"DT", "cowplot" , "colorspace", "PerformanceAnalytics", "corrplot", "Hmisc", "tidyverse", "readxl", "ggpubr", "ggalluvial", "ggstatsplot", "zoo", "stringr", "metagMisc", "ggrepel", "RColorBrewer", "easynls", "ggtree", "ggtext", "treeio")
for (i in RequiredPackages) { #Installs packages if not yet installed
    if (!require(i, character.only = TRUE)) install.packages(i)
}


#Manually install some packages 

#1.ggtree

#if (!requireNamespace("BiocManager", quietly = TRUE))
 #   install.packages("BiocManager")

#BiocManager::install("ggtree")

#2. phyloseq


#if (!requireNamespace("BiocManager", quietly = TRUE))
  #  install.packages("BiocManager")

#BiocManager::install("phyloseq")


library(ggplot2)
  library(Hmisc)
library(easynls)
#library(ggtree)
library(ggtext)
library(glue)
#library(treeio)
library(patchwork)
  library(vegan) 
  library(dplyr) 
  library(scales) 
  library(grid) 
  library(reshape2)
 library(randomcoloR)
#  library(phyloseq) 
  library(magrittr)
  library(geosphere)
  library(matrixStats)
  library(data.table)
  library(DT)
  library(ggalluvial)
  library(cowplot)
  library(colorspace)
  library(corrplot)
  library(tidyverse)
  library(readxl)
  library(corrplot)
  library(ggpubr)
 # library(metagMisc)
  library(RColorBrewer)
library(ggrepel)
library(stringr)
library(zoo)
library("PerformanceAnalytics")
library("ggstatsplot")


```

```{r CTD_data_Lake_profile1, echo=TRUE}

setwd("~/GitHub/Lake_Cadagno_microbial_loop_Saini_et_al_2021/Fig.1-4_Suppl/")


#CTD1 28.08.2017
CTD1_28_Aug_17_R <- read.csv("~/GitHub/Lake_Cadagno_microbial_loop_Saini_et_al_2021/Fig.1-4_Suppl/CTD1_28_Aug_17_II.csv")    

#CTD1 28.08.2017 avg/depth
CTD1_28_Aug_17_avg_R <- read.csv("~/GitHub/Lake_Cadagno_microbial_loop_Saini_et_al_2021/Fig.1-4_Suppl/CTD1_28_Aug_17_avg_II.csv") 

#CTD1 29.08.2017
CTD1_29_Aug_17_R <- read.csv("~/GitHub/Lake_Cadagno_microbial_loop_Saini_et_al_2021/Fig.1-4_Suppl/CTD1_29_Aug_17_II.csv") 

#CTD1 29.08.2017 avg/depth
CTD1_29_Aug_17_avg_R <- read.csv("~/GitHub/Lake_Cadagno_microbial_loop_Saini_et_al_2021/Fig.1-4_Suppl/CTD1_29_Aug_17_avg_II.csv") 

 #CTD2 29.08.2017 avg/depth

 CTD2_29_Aug_17_avg_R <- read.csv("~/GitHub/Lake_Cadagno_microbial_loop_Saini_et_al_2021/Fig.1-4_Suppl/CTD2_29_Aug_17_avg_II.csv")
 
  #CTD2 29.08.2017 not avg_ but raw

 CTD2_29_Aug_17 <- read.csv("~/GitHub/Lake_Cadagno_microbial_loop_Saini_et_al_2021/Fig.1-4_Suppl/CTD2_29_Aug_17.csv")
 
 
 #Metadata
 Meta_data_PLP_VLP_II <-  read.csv("~/GitHub/Lake_Cadagno_microbial_loop_Saini_et_al_2021/Fig.1-4_Suppl/Meta_data_PLP_VLP_II.csv")          
 
 
#Organic nutrients 
  Org_Inrg_R <- read.csv("~/GitHub/Lake_Cadagno_microbial_loop_Saini_et_al_2021/Fig.1-4_Suppl/Org_Inrg_R_II.csv") 
  Org_Inrg_R$H2Smg_L<- c(NA,NA,NA,NA,NA,NA,NA,0,0,0,NA,0,0.09,0,0,0.05,0.37,1.45,2.36, NA, 2.84)
#Secondary production
  profiles_cbp <- read.csv("~/GitHub/Lake_Cadagno_microbial_loop_Saini_et_al_2021/Fig.1-4_Suppl/BP_R.csv")
  
#Primary production
 finalPP_chl23 <- read.csv("~/GitHub/Lake_Cadagno_microbial_loop_Saini_et_al_2021/Fig.1-4_Suppl/PP_R2.csv") 
 
 #Fv/Fm individual
 FvFm_Ind <- read.csv("~/GitHub/Lake_Cadagno_microbial_loop_Saini_et_al_2021/Fig.1-4_Suppl/Fv_Fm_individual_r.csv")
 
#Fv/Fm
 FvFm_R <- read.csv("~/GitHub/Lake_Cadagno_microbial_loop_Saini_et_al_2021/Fig.1-4_Suppl/FvFm_R2.csv")
 


#Fig. 1(B) Turbidity & oxygen
TurbvOxy123<- ggplot(data=CTD1_28_Aug_17_R, aes(y=CTD1_28_Aug_17_R$Depth_m_CTD1_28))+
     geom_path(data=CTD1_28_Aug_17_R, aes(y=CTD1_28_Aug_17_R$Depth_m_CTD1_28, x=CTD1_28_Aug_17_R$Turb_FTU_CTD1_28), 
     color="black", size=2, linetype="dashed")+

     geom_path(data=CTD1_29_Aug_17_R, aes(y=CTD1_29_Aug_17_R$Depth_m_CTD1_29, x=CTD1_29_Aug_17_R$Turb_CTD1_29), 
     color="black", size=2, linetype="solid")+
     geom_path(data=CTD1_29_Aug_17_avg_R, aes(y=as.numeric(CTD1_29_Aug_17_avg_R$Depth), x=CTD1_29_Aug_17_avg_R$Oxy_correc*2.5), 
     color="lightsalmon4", size=2, linetype="solid")+
     geom_path(data=CTD1_28_Aug_17_avg_R, aes(y=as.numeric(CTD1_28_Aug_17_avg_R$Depth), x=CTD1_28_Aug_17_avg_R$DO_mg_l_CTD1_28*2.5), 
     color="lightsalmon4", size=2, linetype="dashed")+

     geom_point(data=CTD1_29_Aug_17_avg_R, aes(y=as.numeric(CTD1_29_Aug_17_avg_R$Depth), x=CTD1_29_Aug_17_avg_R$Oxy_correc*2.5), 
     color="lightsalmon4", size=3.5)+
     geom_point(data=CTD1_28_Aug_17_avg_R, aes(y=as.numeric(CTD1_28_Aug_17_avg_R$Depth), x=CTD1_28_Aug_17_avg_R$DO_mg_l_CTD1_28*2.5), 
     color="lightsalmon4", size=3.5)+
     scale_x_continuous(sec.axis = sec_axis(~./2.5, name = "Oxygen (mg/L)"))+
     xlab("Turbidity (FTU)")+
     ylab("Depth (m)") +
     scale_y_reverse(limits = c(21,1), breaks=c(19, 17, 15.5, 15, 14, 13, 11,9, 7,5, 3, 1))+
     theme_bw()+
     theme(panel.grid.major = element_blank(), panel.grid.minor = element_blank())+
     theme(axis.text=element_text(size=24),axis.title=element_text(size=24,face="bold"))+
     theme(axis.title.x.top = element_text(color = "lightsalmon4"))+
     theme(axis.title.x.bottom = element_text(color = "black"))+
     annotate("rect", xmin=-Inf, xmax=Inf, ymin=-Inf, ymax=12.5, alpha=0.1, fill="green4")+
     annotate("rect", xmin=-Inf, xmax=Inf, ymin=12.5, ymax=13.5, alpha=0.2, fill="deeppink4")+
     annotate("rect", xmin=-Inf, xmax=Inf, ymin=13.5, ymax=15.5, alpha=0.5, fill="deeppink4")+
     annotate("rect", xmin=-Inf, xmax=Inf, ymin=15.5, ymax=16.5, alpha=0.2, fill="deeppink4")+
     annotate("rect", xmin=-Inf, xmax=Inf, ymin=16.5, ymax=Inf, alpha=0.2, fill="pink4")+
     annotate("text", x = 16, y = 2, label = "(- - -) 28 August 2017", colour = "black", size = 8.5)+   
     annotate("text", x = 16, y = 3, label = "( — ) 29 August 2017", colour = "black", size = 8.5)+
     theme(axis.text=element_text(size=24),axis.title=element_text(size=24))

#Fig.1 (C)
Temp_LC_cond123<- ggplot()+
                  geom_path(data=CTD1_29_Aug_17_R, aes(y=CTD1_29_Aug_17_R$Depth_m_CTD1_29,                                                 x=CTD1_29_Aug_17_R$Temp_CTD1_29), 
                  color="black", linetype="solid", size=2)+
                  geom_path(data=CTD1_28_Aug_17_R, aes(y=CTD1_28_Aug_17_R$Depth_m_CTD1_28, x=CTD1_28_Aug_17_R$Temp_CTD1_28), 
                  color="black", linetype="dashed", size=2)+
                  geom_path(data=CTD1_28_Aug_17_R,                                                                                        aes(y=CTD1_28_Aug_17_R$Depth_m_CTD1_28,x=CTD1_28_Aug_17_R$Cond_ms_p_cm_CTD1_28*70),
                  color="lightsalmon4", size=2, linetype="dashed")+

                  geom_path(data=CTD1_29_Aug_17_R, aes(y=CTD1_29_Aug_17_R$Depth_m_CTD1_29,                                                 x=CTD1_29_Aug_17_R$Cond_CTD1_29*70),
                  color="lightsalmon4", size=2, linetype="solid")+
                  scale_x_continuous(sec.axis = sec_axis(~./70, name = "Conductivity (mS/cm)"))+
                  scale_y_reverse(limits = c(21,1), breaks=c(19, 17, 15.5, 15, 14, 13, 11,9, 7,5, 3, 1))+
                  theme_bw()+
                  theme(panel.grid.major = element_blank(), panel.grid.minor = element_blank())+
                  theme(axis.text=element_text(size=24),axis.title=element_text(size=24,face="bold"))+
                  xlab("Temperature (°C)")+
                  theme(axis.title.x.top = element_text(color = "lightsalmon4"))+
                  theme(axis.title.x.bottom = element_text(color = "black"))+
                  ylab("Depth (m)") +
                  theme(axis.title.y = element_blank())+
                  theme(axis.text.y = element_blank())+
                  theme(axis.ticks.y = element_blank())+
                  annotate("rect", xmin=-Inf, xmax=Inf, ymin=-Inf, ymax=12.5, alpha=0.1, fill="green4")+
                  annotate("rect", xmin=-Inf, xmax=Inf, ymin=12.5, ymax=13.5, alpha=0.2, fill="deeppink4")+
                  annotate("rect", xmin=-Inf, xmax=Inf, ymin=13.5, ymax=15.5, alpha=0.5, fill="deeppink4")+
                  annotate("rect", xmin=-Inf, xmax=Inf, ymin=15.5, ymax=16.5, alpha=0.2, fill="deeppink4")+
                  annotate("rect", xmin=-Inf, xmax=Inf, ymin=16.5, ymax=Inf, alpha=0.2, fill="pink4")+
                    theme(axis.text=element_text(size=24),axis.title=element_text(size=24,face="bold"))+
  annotate("text", x = 16, y = 2, label = "(- - -) 28 August 2017", colour = "black", size = 8.5)+   
     annotate("text", x = 16, y = 3, label = "( — ) 29 August 2017", colour = "black", size = 8.5)
  
#Fig.1 (D)
CTD2_29_Aug_17$Depth_m_CTD2_29
chlpar<- ggplot(data=CTD1_29_Aug_17_R, aes(x=Depth_m_CTD1_29)) + 
         geom_path(data=CTD1_29_Aug_17_R, aes(y= as.numeric(CTD1_29_Aug_17_R$Depth_m_CTD1_29),               x=CTD1_29_Aug_17_R$Chl_A_CTD1_29*10), color="lightsalmon4", linetype="solid",  size=2)+
       geom_path(data=CTD1_28_Aug_17_R, aes(y=as.numeric(CTD1_28_Aug_17_R$Depth_m_CTD1_28),x=CTD1_28_Aug_17_R$Chl_A_ug_p_l_CTD1_28*10), color="lightsalmon4", linetype="dashed", size=2)+
   geom_path(data=CTD2_29_Aug_17, aes(y=as.numeric(Depth_m_CTD2_29), x=CTD2_29_Aug_17$Phycocyanin_CTD2_29*10), color="lightseagreen", linetype="solid", size=1.5)+
  # geom_point(data=CTD2_29_Aug_17, aes(y=as.numeric(Depth_m_CTD2_29), x=CTD2_29_Aug_17$Phycocyanin_CTD2_29*10), color="lightsalmon1", size=3.5)+
       geom_path(data=CTD2_29_Aug_17_avg_R, aes(y=as.numeric(CTD2_29_Aug_17_avg_R$Depth), x=CTD2_29_Aug_17_avg_R$PAR_CTD2_29),
       linetype="solid", size=2, color="black")+
       geom_point(data=CTD2_29_Aug_17_avg_R, aes(y=as.numeric(CTD2_29_Aug_17_avg_R$Depth),x=CTD2_29_Aug_17_avg_R$PAR_CTD2_29)        , color="black", size=3.5)+
       scale_y_reverse(limits = c(21,1), breaks=c(19, 17.0, 15.5,15,14, 13.0, 11.0,9.0, 7.0,5.0, 3.0, 1))+
       scale_x_continuous(limits = c(-10,600), sec.axis = sec_axis(~./10, name = "Photosynthetic Pigments (μg/L)"))+
       xlab(expression( bold("PAR "("w/m" ^-2))))+  ##Superscript example  xlab(expression(paste("DOC (mg ", L^-1,")")))
       ylab("Depth(m)")+
       theme_bw()+
       theme(axis.text=element_text(size=24),axis.title=element_text(size=24,face="bold"))+
       annotate("rect", xmin=-Inf, xmax=Inf, ymin=-Inf, ymax=12.5, alpha=0.1, fill="green4")+
       annotate("rect", xmin=-Inf, xmax=Inf, ymin=12.5, ymax=13.5, alpha=0.2, fill="deeppink4")+
       annotate("rect", xmin=-Inf, xmax=Inf, ymin=13.5, ymax=15.5, alpha=0.5, fill="deeppink4")+
       annotate("rect", xmin=-Inf, xmax=Inf, ymin=15.5, ymax=16.5, alpha=0.2, fill="deeppink4")+
       annotate("rect", xmin=-Inf, xmax=Inf, ymin=16.5, ymax=Inf, alpha=0.2, fill="pink4")+
       #theme(plot.title = element_text(hjust = 0.5))+
       theme(axis.title.y = element_blank())+
       theme(axis.text.y = element_blank())+
       theme(axis.ticks.y = element_blank())+ theme(axis.line.y.right = element_blank())+
       theme(panel.grid.major = element_blank(), panel.grid.minor = element_blank())+
       theme(axis.title.x.top = element_text(color = "lightsalmon4"))+
       theme(axis.title.x=element_text(size=24,face="bold"))+
  annotate("text", x = 16, y = 2, label = "(- - -) 28 August 2017", colour = "black", size = 8.5)+   
     annotate("text", x = 16, y = 3, label = "( — ) 29 August 2017", colour = "black", size = 8.5)

    #Legend annotation with Adobe Illustrator

#Fig.1 (E)
 H2S_Amm<- ggplot(data=Org_Inrg_R, aes(y=depth_m))+
    geom_path(data=Org_Inrg_R, aes(y= depth_m, x = Ammonium_ug_L), colour="black", size=2)+
      geom_point(data=Org_Inrg_R, aes(y= depth_m, x = Ammonium_ug_L), colour="black", size=3.5)+
    geom_point(data=Org_Inrg_R, aes(y=depth_m, x=H2Smg_L*100), color="lightsalmon4", size=3.5)+
    geom_path(data=Org_Inrg_R[!is.na(Org_Inrg_R$H2Smg_L*100),], aes(y=depth_m, x=H2Smg_L*100), color="lightsalmon4", size=2, linetype="dashed")+
  scale_x_continuous(sec.axis = sec_axis(~./100, name = "Hydrogen Sulphide (mg/L)"))+
   scale_y_reverse(limits = c(21,1), breaks=c(19, 17, 15.5, 15, 14, 13, 11,9, 7,5, 3, 1))+
    theme_bw()+
     theme(panel.grid.major = element_blank(), panel.grid.minor = element_blank())+
              theme(axis.text=element_text(size=24),axis.title=element_text(size=24,face="bold"))+
    xlab("Ammonium (µg/L)")+
    theme(axis.title.x.top = element_text(color = "lightsalmon4"))+
    theme(axis.title.x.bottom = element_text(color = "black"))+
    ylab("Depth (m)") +
   # theme(axis.title.y = element_blank())+
  #  theme(axis.text.y = element_blank())+
  #  theme(axis.ticks.y = element_blank())+
     annotate("rect", xmin=-Inf, xmax=Inf, ymin=-Inf, ymax=12.5, alpha=0.1, fill="green4")+
   annotate("rect", xmin=-Inf, xmax=Inf, ymin=12.5, ymax=13.5, alpha=0.2, fill="deeppink4")+
  annotate("rect", xmin=-Inf, xmax=Inf, ymin=13.5, ymax=15.5, alpha=0.5, fill="deeppink4")+
    annotate("rect", xmin=-Inf, xmax=Inf, ymin=15.5, ymax=16.5, alpha=0.2, fill="deeppink4")+
    annotate("rect", xmin=-Inf, xmax=Inf, ymin=16.5, ymax=Inf, alpha=0.2, fill="pink4")+
   annotate("segment", x = 0, xend = 0, y =5, yend = 9,
  colour = "lightsalmon4", size=2, linetype = "dashed")+
   annotate("text", x = 16, y = 2, label = "(- - -) 28 August 2017", colour = "black", size = 8.5)+   
     annotate("text", x = 16, y = 3, label = "( — ) 29 August 2017", colour = "black", size = 8.5)
 
#Fig.1 (F) 
 FvFm_R <- read.csv("~/GitHub/Lake_Cadagno_microbial_loop_Saini_et_al_2021/Fig.1-4_Suppl/FvFm_R2.csv")
 

 
 
 finalPP_chl23$net_pp
 #Try 2
  shsh2<- ggplot(data=finalPP_chl23, aes(x=Depth)) + theme_bw()+
  geom_point(data=finalPP_chl23, aes(y = log(net_pp), colour="Net"), size=3.5)+
  geom_line(data=finalPP_chl23, aes(y = log(net_pp), x=Depth),linetype = "dashed", colour="darkolivegreen4", size=2)+
    geom_point(profiles_cbp, mapping= aes(y=log(profiles_cbp$BP_ng), x=profiles_cbp$depth_m, color= "SP"), size=3.5)+
    geom_errorbar(data=profiles_cbp, mapping= aes(ymin=log(BP_ng-SD_BP), ymax=log(BP_ng+SD_BP), x=profiles_cbp$depth_m), width=.3,position=position_dodge(0.1), inherit.aes = FALSE)+
  geom_point(data=FvFm_R, mapping= aes(y=FvFm_R$Fv.Fm.ave*10, x= as.numeric(FvFm_R$depth..m.), color= "Fv/Fm"), size=3.5)+
     geom_errorbar(data=FvFm_R, mapping= aes(ymin=(Fv.Fm.ave-std)*10, ymax=(Fv.Fm.ave+std)*10, x=FvFm_R$depth..m.), width=.3,position=position_dodge(0.1), inherit.aes = FALSE)+
    scale_y_continuous(sec.axis = sec_axis(~./10, name = "Fv/Fm"))+
    scale_x_reverse(limits = c(21,1), breaks=c(19, 17, 15.5, 15,14, 13, 11, 9, 7,5, 3, 1))+
  theme(legend.title=element_text(size=22), legend.text = element_text(size=20, face = "bold"))+
  coord_flip()+
        theme(legend.position = c(0.25, 0.15), legend.direction = "vertical",
                   legend.background = element_rect(size=0.3, 
                                   colour ="transparent", fill = "transparent"), legend.text = element_text(color = "black", size = 22))+
  annotate("rect", ymin=-Inf, ymax=Inf, xmin=-Inf, xmax=12.5, alpha=0.1, fill="green4")+
     annotate("rect", ymin=-Inf, ymax=Inf, xmin=12.5, xmax=13.5, alpha=0.2, fill="deeppink4")+
     annotate("rect", ymin=-Inf, ymax=Inf, xmin=13.5, xmax=15.5, alpha=0.5, fill="deeppink4")+
     annotate("rect", ymin=-Inf, ymax=Inf, xmin=15.5, xmax=16.5, alpha=0.2, fill="deeppink4")+
     annotate("rect", ymin=-Inf, ymax=Inf, xmin=16.5, xmax=Inf, alpha=0.2, fill="pink4")+
       theme(axis.text=element_text(size=24),axis.title=element_text(size=24,face="bold"))+
  coord_flip()  + ylab("Log of PP & SP (ngC L-1 H-1)") + 
  theme(
  # Hide panel borders and remove grid lines
  panel.grid.major = element_blank(),
  panel.grid.minor = element_blank(),
  # Change axis line 
  )+
  theme(axis.title.y = element_blank())+
    theme(axis.text.y = element_blank())+
    theme(axis.ticks.y = element_blank())+
       theme(axis.title.x.top = element_text(color = "lightsalmon4"))+
 theme(legend.key = element_rect(colour = "transparent", fill = "transparent"))+
                  theme(axis.text=element_text(size=24),axis.title=element_text(size=24,face="bold"))+
   scale_colour_manual("", breaks = c("Fv/Fm", "Net", "SP"),
  values = c("lightsalmon4", "darkolivegreen4", "black"), labels=c("Fv/Fm", "NPP", "SP"))+
  geom_segment(aes(x = 1 , y = 1.835886, xend = 2, yend = 1.730184), linetype="dashed", color = "black", size=2)+
  geom_segment(aes(x = 2 , y = 1.730184, xend = 3, yend = 1.783575), linetype="dashed", color = "black", size=2)+
  geom_segment(aes(x = 3 , y = 1.783575, xend = 4, yend = 1.647750), linetype="dashed", color = "black", size=2)+
  geom_segment(aes(x = 4 , y = 1.647750, xend = 5, yend = 1.884905), linetype="dashed", color = "black", size=2)+
  geom_segment(aes(x = 5 , y = 1.884905, xend = 6, yend = 3.250048), linetype="dashed", color = "black", size=2)+
  geom_segment(aes(x = 6 , y = 3.250048, xend = 7, yend = 2.459175), linetype="dashed", color = "black", size=2)+
  geom_segment(aes(x = 7 , y = 2.459175, xend = 8, yend =  3.401523), linetype="dashed", color = "black", size=2)+
  geom_segment(aes(x = 8 , y = 3.401523, xend = 9, yend = 3.761246), linetype="dashed", color = "black", size=2)+
  geom_segment(aes(x = 9 , y = 3.761246, xend = 10, yend = 3.590435), linetype="dashed", color = "black", size=2)+
  geom_segment(aes(x = 10 , y = 3.590435, xend = 11, yend = 3.576309), linetype="dashed", color = "black", size=2)+
  geom_segment(aes(x = 11 , y = 3.576309, xend = 13, yend = 4.285812), linetype="dashed", color = "black", size=2)+
    geom_segment(aes(x = 13 , y = 4.285812, xend = 13.5, yend = 6.009835), linetype="solid", color = "black", size=2)+
  geom_segment(aes(x = 13.5 , y = 6.009835, xend = 14, yend = 5.945037), linetype="solid", color = "black", size=2)+
    geom_segment(aes(x = 14 , y = 5.945037, xend = 14.5, yend = 5.988719), linetype="solid", color = "black", size=2)+
    geom_segment(aes(x = 14.5 , y = 5.988719, xend = 15, yend = 6.179743), linetype="solid", color = "black", size=2)+
    geom_segment(aes(x = 15 , y = 6.179743, xend = 15.5, yend = 6.207229), linetype="solid", color = "black", size=2)+
    geom_segment(aes(x = 15.5 , y = 6.207229, xend = 16, yend = 5.957780), linetype="solid", color = "black", size=2)+
    geom_segment(aes(x = 16 , y = 5.957780, xend = 17, yend = 5.55177), linetype="solid", color = "black", size=2)+
    geom_segment(aes(x = 17 , y = 5.55177, xend = 19, yend = 5.43806), linetype="solid", color = "black", size=2)+
  theme(legend.box.background = element_rect(colour = "black",   fill = "transparent"))+
  theme(legend.title = element_blank()) +
      geom_segment(aes(x = 1 , y = 0.4046078*10, xend = 2, yend = 0.5596724*10), linetype="dashed", color = "lightsalmon4", size=2)+
  geom_segment(aes(x = 2 , y = 0.5596724*10, xend = 3, yend = 0.5416852*10), linetype="dashed", color = "lightsalmon4", size=2)+
  geom_segment(aes(x = 3 , y = 0.5416852*10, xend = 4, yend = 0.5512007*10), linetype="dashed", color = "lightsalmon4", size=2)+
  geom_segment(aes(x = 4 , y = 0.5512007*10, xend = 5, yend = 0.5183846*10), linetype="dashed", color = "lightsalmon4", size=2)+
  geom_segment(aes(x = 5 , y = 0.5183846*10, xend = 6, yend = 0.5503139*10), linetype="dashed", color = "lightsalmon4", size=2)+
  geom_segment(aes(x = 6 , y = 0.5503139*10, xend = 7, yend = 0.5669652*10), linetype="dashed", color = "lightsalmon4", size=2)+
  geom_segment(aes(x = 7 , y = 0.5669652*10, xend = 8, yend =  0.5530409*10), linetype="dashed", color = "lightsalmon4", size=2)+
  geom_segment(aes(x = 8 , y = 0.5530409*10, xend = 9, yend = 0.5340161*10), linetype="dashed", color = "lightsalmon4", size=2)+
  geom_segment(aes(x = 9 , y = 0.5340161*10, xend = 10, yend = 0.5198984*10), linetype="dashed", color = "lightsalmon4", size=2)+
  geom_segment(aes(x = 10 , y = 0.5198984*10, xend = 11, yend = 0.5258582*10), linetype="dashed", color = "lightsalmon4", size=2)+
  geom_segment(aes(x = 11 , y = 0.5258582*10, xend = 13, yend = 0.5249207*10), linetype="dashed", color = "lightsalmon4", size=2)+
    geom_segment(aes(x = 13 , y = 0.5249207*10, xend = 13.5, yend = 0.4877256*10), linetype="solid", color = "lightsalmon4", size=2)+
  geom_segment(aes(x = 13.5 , y = 0.4877256*10, xend = 14, yend = 0.4551373*10), linetype="solid", color = "lightsalmon4", size=2)+
    geom_segment(aes(x = 14 , y = 0.4551373*10, xend = 14.5, yend = 0.4055380*10), linetype="solid", color = "lightsalmon4", size=2)+
    geom_segment(aes(x = 14.5 , y = 0.4055380*10, xend = 15, yend = 0.3564647*10), linetype="solid", color = "lightsalmon4", size=2)+
    expand_limits(x = 0, y = 0)+
     annotate("text", x = 16, y = 2, label = "(- - -) 28 August 2017", colour = "black", size = 8.5)+   
     annotate("text", x = 16, y = 3, label = "( — ) 29 August 2017", colour = "black", size = 8.5)
  

   
  
  
  
 profiles_cbp$BP_nglog <- log(profiles_cbp$BP_ng)

 
 #Fig.1 (G) 
  profiles_POC<- Org_Inrg_R[!is.na(Org_Inrg_R$POC),]
   DOC<- ggplot(profiles_POC, aes(x=DOC, y=depth_m))+
    theme_bw()+
      geom_point(size=3.5)+
     geom_path(size=2, linetype="solid")+
   
    #    scale_x_continuous(sec.axis = sec_axis(~.*1, name = "SiO2 (mg/L)"))+
  #  geom_path(data=profiles_POC, aes(x=SiO2_mg_L/1, y=depth_m), color="lightsalmon4", size=2, linetype="solid")+
   #     geom_point(data=profiles_POC, aes(x=SiO2_mg_L/1, y=depth_m), color="lightsalmon4", size=3.5)+
    xlab("DOC (ppm)") +
    ylab("Depth (m)") +
    geom_point(color= "black", size=3.5)+ 
      scale_y_reverse(limits = c(21,1), breaks=c(19, 17, 15.5, 15,14, 13, 11,9, 7,5, 3, 1))+
    annotate("rect", xmin=-Inf, xmax=Inf, ymin=-Inf, ymax=12.5, alpha=0.1, fill="green4")+
    annotate("rect", xmin=-Inf, xmax=Inf, ymin=12.5, ymax=13.5, alpha=0.2, fill="deeppink4")+
    annotate("rect", xmin=-Inf, xmax=Inf, ymin=13.5, ymax=15.5, alpha=0.5, fill="deeppink4")+
    annotate("rect", xmin=-Inf, xmax=Inf, ymin=15.5, ymax=16.5, alpha=0.2, fill="deeppink4")+
    annotate("rect", xmin=-Inf, xmax=Inf, ymin=16.5, ymax=Inf, alpha=0.2, fill="pink4")+
    #geom_area(fill = "black", linetype="dashed", position = "left")+
   theme_bw()+ 
      theme(axis.text=element_text(size=24),axis.title=element_text(size=24,face="bold"))+
      theme(panel.grid.major = element_blank(),
          panel.grid.minor = element_blank(), axis.line = element_line(colour = "black"))+
    # theme(axis.title.y = element_blank())+
    #theme(axis.text.y = element_blank())+
    #theme(axis.ticks.y = element_blank())+
         theme(axis.text.x.top = element_text(color = "black"))+
     theme(axis.title.x.top = element_text(color = "lightsalmon4"))+
        geom_segment(aes(x = 2.0580 , y = 17, xend = 2.05, yend = 19), linetype="solid", color = "black", size=2)+
       theme(axis.title.y = element_blank())+
    theme(axis.text.y = element_blank())+
    theme(axis.ticks.y = element_blank())+
     
         scale_x_continuous(sec.axis = sec_axis(~.*2, name = "POC (ppm)"))+
       geom_point(data=profiles_POC, aes(y=depth_m, x=POC/2), color="lightsalmon4", size=3.5)+
    geom_path(data=profiles_POC[!is.na(profiles_POC$POC),], aes(y=depth_m, x=POC/2), color="lightsalmon4", size=2, linetype="solid")+
              theme(axis.title.x.top = element_text(color = "lightsalmon4"))+
     annotate("text", x = 3, y = 2, label = "(- - -) 28 August 2017", colour = "black", size = 8.5)+   
     annotate("text", x = 3, y = 3, label = "( — ) 29 August 2017", colour = "black", size = 8.5)
  
   
   
   #Fig.1 (H) 

   
    profiles_PON<- Org_Inrg_R[!is.na(Org_Inrg_R$PON),]
        profiles_POS<- Org_Inrg_R[!is.na(Org_Inrg_R$POS),]


   PON<- ggplot(profiles_PON, aes(x=PON, y=depth_m))+
    geom_point() + theme_bw()+
    geom_path(na.rm = T) +
   # ggtitle("E. Particulate Organic Nitrogen") +
    xlab("PON (ppm)") +
    ylab("") +
    geom_point(color= "black", size=3.5)+ 
    geom_path(size=2)+
    #geom_area(fill = "black", linetype="dashed", position = "left")+
      scale_y_reverse(limits = c(21,1), breaks=c(19, 17, 15, 13, 11,9, 7,5, 3, 1))+
    annotate("rect", xmin=-Inf, xmax=Inf, ymin=-Inf, ymax=12.5, alpha=0.1, fill="green4")+
    annotate("rect", xmin=-Inf, xmax=Inf, ymin=12.5, ymax=13.5, alpha=0.2, fill="deeppink4")+
    annotate("rect", xmin=-Inf, xmax=Inf, ymin=13.5, ymax=15.5, alpha=0.5, fill="deeppink4")+
    annotate("rect", xmin=-Inf, xmax=Inf, ymin=15.5, ymax=16.5, alpha=0.2, fill="deeppink4")+
    annotate("rect", xmin=-Inf, xmax=Inf, ymin=16.5, ymax=Inf, alpha=0.2, fill="pink4")+
    #geom_area(fill = "black", linetype="dashed", position = "left")+
   theme_bw()+ theme(panel.border = element_rect(colour = "black", fill=NA))+
    theme(axis.title.y = element_blank())+
    theme(axis.text.y = element_blank())+
    theme(axis.ticks.y = element_blank())+
      theme(axis.text=element_text(size=24),axis.title=element_text(size=24,face="bold"))+
      theme(panel.grid.major = element_blank(),
          panel.grid.minor = element_blank(), axis.line = element_line(colour = "black"))+
       scale_x_continuous(sec.axis = sec_axis(~.*0.65, name = "POS (ppm)"))+
      theme(axis.title.x.top = element_text(color = "lightsalmon4"))+
    theme(axis.title.x.bottom = element_text(color = "black"))+
      geom_path(data=profiles_POS, aes(y= depth_m, x = POS/0.65), colour="lightsalmon4", size=2)+
      geom_point(data=profiles_POS, aes(y= depth_m, x = POS/0.65), colour="lightsalmon4", size=3.5)+
     annotate("text", x = 0.50, y = 2, label = "(- - -) 28 August 2017", colour = "black", size = 8.5)+   
     annotate("text", x = 0.50, y = 3, label = "( — ) 29 August 2017", colour = "black", size = 8.5)
  
 
 
figure1<- (plot_spacer() | TurbvOxy123| Temp_LC_cond123| chlpar) / ( H2S_Amm | shsh2| DOC | PON )  +
     plot_annotation(tag_levels = 'A', , tag_prefix = '(',
                         tag_suffix = ')') & 
  theme(plot.tag = element_text(size = 26, face = "bold")) #23x23

figure1

figure1_gg<- ggarrange(NULL, TurbvOxy123, Temp_LC_cond123, chlpar, ncol=4, widths = c(1,1.20,1,1), align="h") 

figure1b_gg<- ggarrange(H2S_Amm, shsh2,  DOC, PON, ncol=4, widths = c(1.20,1,1,1))

figure1_gg / figure1b_gg + 
  plot_annotation(tag_levels = 'A', , tag_prefix = '(',
                         tag_suffix = ')') & 
  theme(plot.tag = element_text(size = 26, face = "bold"))
  #23x25 pdf ###Figure 1







```

```{r fig.2, echo=TRUE}

#PLP and VLP R file
plp_vlp <- read.csv("~/GitHub/Lake_Cadagno_microbial_loop_Saini_et_al_2021/Fig.1-4_Suppl/PLP_VLP_R.csv")
plp_vlp$TCell_cnt_var<- plp_vlp$TCell_cnt_SD*plp_vlp$TCell_cnt_SD
plp_vlp$water_layer<- c("Mixolimnion","Mixolimnion","Mixolimnion","Mixolimnion","Mixolimnion","Mixolimnion","Mixolimnion","Mixolimnion","Mixolimnion","Mixolimnion","Mixolimnion","Mixolimnion", "Chemocline", "Chemocline", "Chemocline", "Chemocline", "Chemocline", "Monimolimnion", "Monimolimnion", "Monimolimnion", "Monimolimnion")

#Checking mean, variance, sd according to mixolimnion, chemocline, monimolinion
  plp_vlp2<- plp_vlp %>% 
    group_by(water_layer) %>% 
     dplyr:: summarise_all(list(mean))
plp_vlp2$varsd<- sqrt(plp_vlp2$TCell_cnt_var)

#Fig. 2(A)
     total_cell<- ggplot(plp_vlp, aes(x=as.numeric(Depth), y=TCell_cnt_ml)) + 
    #geom_line(size=2) +
    geom_point(size=3.5)+
  geom_errorbar(data=plp_vlp, aes(ymin=TCell_cnt_ml-TCell_cnt_SD, ymax=TCell_cnt_ml+TCell_cnt_SD), width=.2,
                  position=position_dodge(0.05))+
    scale_x_reverse(limits = c(21,1), breaks=c(19, 17, 15.5, 15, 14, 13, 11,9, 7,5, 3, 1))+
   # ggtitle("A. Total Cell Count")+
    ylab("Total Cell count/mL")+ 
    xlab("Depth (m)")+
    theme(axis.text=element_text(size=24),
          axis.title=element_text(size=24,face="bold"))+
    scale_y_continuous(limits = c(0, 1000000), labels = scales::scientific)+
  coord_flip() + theme_bw()+
    annotate("rect", ymin=-Inf, ymax=Inf, xmin=-Inf, xmax=12.5, alpha=0.1, fill="green4")+
    annotate("rect", ymin=-Inf, ymax=Inf, xmin=12.5, xmax=13.5, alpha=0.2, fill="deeppink4")+
    annotate("rect", ymin=-Inf, ymax=Inf, xmin=13.5, xmax=15.5, alpha=0.5, fill="deeppink4")+
    annotate("rect", ymin=-Inf, ymax=Inf, xmin=15.5, xmax=16.5, alpha=0.2, fill="deeppink4")+
    annotate("rect", ymin=-Inf, ymax=Inf, xmin=16.5, xmax=Inf, alpha=0.2, fill="pink4")+
    theme(axis.text=element_text(size=24),axis.title=element_text(size=24,face="bold"))+
    theme(axis.line = element_line(color = "black"))+
    scale_x_reverse(limits = c(21,1), breaks=c(17, 15, 15.5,14, 13, 11,9, 7,5, 3, 1))+
    theme(legend.position = "none")+
     theme(axis.text=element_text(size=24),
          axis.title=element_text(size=24,face="bold"))+
    theme(
  # Hide panel borders and remove grid lines
  panel.grid.major = element_blank(),
  panel.grid.minor = element_blank(),
  # Change axis line
  )+ geom_segment(aes(x = 1 , y = 104666.67, xend = 2, yend = 106000), linetype="dashed", color = "black", size=2)+
    geom_segment(aes(x = 2 , y = 106000, xend = 3, yend = 95666), linetype="dashed", color = "black", size=2)+
    geom_segment(aes(x = 3 , y = 95666, xend = 4, yend = 164000), linetype="dashed", color = "black", size=2)+
    geom_segment(aes(x = 4 , y = 164000, xend = 5, yend =  86666), linetype="dashed", color = "black", size=2)+
    geom_segment(aes(x = 5 , y = 86666, xend = 6, yend = 150000), linetype="dashed", color = "black", size=2)+
    geom_segment(aes(x = 6 , y = 150000, xend = 7, yend = 157333), linetype="dashed", color = "black", size=2)+
    geom_segment(aes(x = 7 , y = 157333, xend = 8, yend = 74666), linetype="dashed", color = "black", size=2)+
    geom_segment(aes(x = 8 , y = 74666, xend = 9, yend = 73000), linetype="dashed", color = "black", size=2)+
    geom_segment(aes(x = 9 , y = 73000, xend =10, yend = 87666), linetype="dashed", color = "black", size=2)+
    geom_segment(aes(x = 10 , y = 87666, xend = 11, yend = 104333), linetype="dashed", color = "black", size=2)+
    geom_segment(aes(x = 11 , y = 104333, xend = 12, yend = 110666), linetype="dashed", color = "black", size=2)+
    geom_segment(aes(x = 12 , y = 110666, xend = 13, yend = 95333), linetype="solid", color = "black", size=2)+
    geom_segment(aes(x = 13 , y = 95333, xend = 13.5, yend = 152666), linetype="dashed", color = "black", size=2)+
    geom_segment(aes(x = 13.5 , y = 152666, xend = 14, yend = 742000), linetype="solid", color = "black", size=2)+
    geom_segment(aes(x = 14 , y = 742000, xend = 15, yend = 905000), linetype="solid", color = "black", size=2)+
    geom_segment(aes(x = 15 , y = 905000, xend = 15.5, yend = 585000), linetype="solid", color = "black", size=2)+
        geom_segment(aes(x = 15.5 , y = 585000, xend = 16, yend = 461000), linetype="solid", color = "black", size=2)+
        geom_segment(aes(x = 16 , y = 461000, xend = 17, yend = 434666), linetype="solid", color = "black", size=2)+
        geom_segment(aes(x = 17 , y = 434666, xend = 18, yend = 450666), linetype="solid", color = "black", size=2)+
        geom_segment(aes(x = 18 , y = 450666, xend = 19, yend = 397000), linetype="solid", color = "black", size=2)
     

#Fig. 2(B)
FlowVLP78abc<- ggplot(plp_vlp, aes(x=as.numeric(Depth), y=VLP_p_ml)) + 
    #geom_line(size=2) +
    geom_point(size=3.5)+
  geom_errorbar(data=plp_vlp, aes(ymin=VLP_p_ml-VLP_sd_ml, ymax=VLP_p_ml+VLP_sd_ml), width=.2,
                  position=position_dodge(0.05))+
    scale_x_reverse(limits = c(21,1), breaks=c(19, 17, 15.5, 15, 14, 13, 11,9, 7,5, 3, 1))+
  coord_flip()+
  theme_bw()+

    annotate("rect", ymin=-Inf, ymax=Inf, xmin=-Inf, xmax=12.5, alpha=0.1, fill="green4")+
    annotate("rect", ymin=-Inf, ymax=Inf, xmin=12.5, xmax=13.5, alpha=0.2, fill="deeppink4")+
    annotate("rect", ymin=-Inf, ymax=Inf, xmin=13.5, xmax=15.5, alpha=0.5, fill="deeppink4")+
    annotate("rect", ymin=-Inf, ymax=Inf, xmin=15.5, xmax=16.5, alpha=0.2, fill="deeppink4")+
    annotate("rect", ymin=-Inf, ymax=Inf, xmin=16.5, xmax=Inf, alpha=0.2, fill="pink4")+
  theme(
  # Hide panel borders and remove grid lines
  panel.grid.major = element_blank(),
  panel.grid.minor = element_blank(),
  # Change axis line
  )+
   ylab("Virus-like-particles/mL")+
    xlab("Depth (m)")+
    #ggtitle("A. Abundance of VLPs")+
  theme(axis.text=element_text(size=21.5),
          axis.title=element_text(size=24,face="bold"))+
  #theme(axis.title.y = element_blank(), axis.text.y = element_blank(), axis.ticks.y= element_blank())+
 geom_segment(aes(x = 3 , y = 72734310.2367783, xend = 5, yend = 70269433.106576), linetype="dashed", color = "black", size=2)+

    geom_segment(aes(x = 5 , y= 70269433.106576, xend = 11, yend = 171345477.337887), linetype="dashed", color = "black", size=2)+
    geom_segment(aes(x = 11 , y = 171345477.337887, xend = 13, yend = 152087893.881184), linetype="dashed", color = "black", size=2)+
    geom_segment(aes(x = 13 , y = 152087893.881184, xend = 15, yend = 142709765.825333), linetype="solid", color = "black", size=2)+
    geom_segment(aes(x = 15 , y = 142709765.825333, xend = 15.5, yend = 148248347.274825), linetype="solid", color = "black", size=2)+
    geom_segment(aes(x = 15.5 , y = 148248347.274825, xend = 17, yend = 140365461.613217), linetype="solid", color = "black", size=2)+
     geom_segment(aes(x = 17 , y = 140365461.613217, xend = 19, yend = 143954609.653385), linetype="solid", color = "black", size=2, arrow = arrow(type = "open", angle = 30, length = unit(0.15, "inches")))+
       theme(axis.text=element_text(size=24),axis.title=element_text(size=24))+
   theme(axis.title.y = element_blank())+
    theme(axis.text.y = element_blank())+
    theme(axis.ticks.y = element_blank())


#Fig. 2(C)
vmr2<- ggplot()+ theme_bw()+
  geom_point(data=subset(plp_vlp, !is.na(VMR)), aes(y=VMR, x=as.numeric(Depth)), size=3.5, color= "black")+
     geom_segment(aes(x = 3 , y = 760.28, xend = 5, yend = 810.801), linetype="dashed", color = "black", size=2)+
      geom_segment(aes(x = 5 , y = 810.801, xend = 11, yend = 1642.288), linetype="dashed", color = "black", size=2)+
    geom_segment(aes(x = 11 , y = 1642.288, xend = 13, yend = 1595.327), linetype="dashed", color = "black", size=2)+
     geom_segment(aes(x = 13 , y = 1595.327, xend = 15, yend = 157.690), linetype="solid", color = "black", size=2)+
    geom_segment(aes(x = 15 , y = 157.690, xend = 15.5, yend = 253.416), linetype="solid", color = "black", size=2)+
     geom_segment(aes(x = 15.5 , y = 253.416, xend = 17, yend = 322.926), linetype="solid", color = "black", size=2)+
     geom_segment(aes(x = 17 , y = 322.926, xend = 19, yend = 362.606), linetype="solid", color = "black", size=2, arrow = arrow(type = "open", angle = 30, length = unit(0.15, "inches")))+
     theme(axis.title.x.top = element_text(color = "black"))+
    theme(axis.text=element_text(size=24),
          axis.title=element_text(size=24,face="bold"))+
  coord_flip()+
     theme(axis.title.y = element_blank())+
    theme(axis.text.y = element_blank())+
    theme(axis.ticks.y = element_blank())+
  scale_x_reverse(limits = c(21,1), breaks=c(19, 17, 15.5, 15, 14, 13, 11,9, 7,5, 3, 1))+
  theme(
  # Hide panel borders and remove grid lines
  panel.grid.major = element_blank(),
  panel.grid.minor = element_blank(),
  # Change axis line
  )+
   annotate("rect", ymin=-Inf, ymax=Inf, xmin=-Inf, xmax=12.5, alpha=0.1, fill="green4")+
    annotate("rect", ymin=-Inf, ymax=Inf, xmin=12.5, xmax=13.5, alpha=0.2, fill="deeppink4")+
    annotate("rect", ymin=-Inf, ymax=Inf, xmin=13.5, xmax=15.5, alpha=0.5, fill="deeppink4")+
    annotate("rect", ymin=-Inf, ymax=Inf, xmin=15.5, xmax=16.5, alpha=0.2, fill="deeppink4")+
    annotate("rect", ymin=-Inf, ymax=Inf, xmin=16.5, xmax=Inf, alpha=0.2, fill="pink4")+
  ylab("Virus-to-microbe-ratio (VMR)")


fig.2<- (total_cell | FlowVLP78abc | vmr2) +
    plot_annotation(tag_levels = 'A', tag_prefix = '(',
                         tag_suffix = ')') & 
  theme(plot.tag = element_text(size = 26, face = "bold"))
fig.2

```

```{r fig.3, echo=TRUE}

set.seed(8363830)

#Absolute Abundance at Genus 
AA_Genus <- read.csv("~/GitHub/Lake_Cadagno_microbial_loop_Saini_et_al_2021/16s_Files_R_codes/SI_Table2.csv")

#removing cells <1000 
AA_Genus2<- AA_Genus[which(AA_Genus$AA_Arch_Bac>=100), ]


 # uahbs234ab_cyna_CPF2F<- AA_Genus[which(AA_Genus$depth_m=="15"), ]
#  uahbs234ab_cyna_CPF2FB<- uahbs234ab_cyna_CPF2F[which(uahbs234ab_cyna_CPF2F$Phylum=="Cyanobacteria"), ]
 # sum(uahbs234ab_cyna_CPF2FB$Abundance_P)

#Absolute Abundance at Phylum 
#gpsfSI2 <- read.csv("~/GitHub/Lake_Cadagno_microbial_loop_Saini_et_al_2021/Fig.1-4_Suppl/SI_Table2.csv")






#Fig 3(A)
jascolor1<-    c("steelblue","#56B4E9", "lightblue1","burlywood1","burlywood3", "palevioletred2", "hotpink1", "hotpink3", "chartreuse3", "palegreen",   "snow2", "snow3", "snow4", "dimgray","tan3", "tan2", "tan", "thistle1", "thistle2", "plum1")  #Picking colors
#"chartreuse3", "chartreuse4""plum3","plum4"


 yv1a_CC <-        ggplot(data = AA_Genus2,
                            mapping = aes_string(x ="Sample",fill="Phylum",y="AA_Arch_Bac"))+
    geom_bar(stat="identity")+
    theme(legend.position="none")+
    ylab("Absolute Abundance (>100 Cells/mL)")+
    coord_flip()+                   
    theme_bw()+
    xlab("Depth (m)")+
    scale_x_discrete(labels=c("LC-19m-E1"="19","LC-17m-E1"="17",
                              "LC-15point5m-E1"="15.5", 
                              "Cadagno15m-E2"="15", "LC-13m-E1"="13", "LC-11m-E1"="11","LC-9m-E1"="9",
                              "Cadagno7m-E2"="7", "LC-5m-E1"="5", "LC-3m-E1"="3"), 
                      limits=c("", "LC-19m-E1","LC-17m-E1"," ","LC-15point5m-E1", "Cadagno15m-E2",
                           "LC-13m-E1"," ", "LC-11m-E1", "LC-9m-E1", "Cadagno7m-E2",
                              "LC-5m-E1", "LC-3m-E1", "")) +  
    scale_fill_manual(values = (jascolor1))+
    theme(legend.box.background = element_rect(color="white", size=2), panel.border = element_rect(colour = "black", fill=NA, size=2))+
      theme(axis.text = element_text(size = 12))+
     guides(fill=guide_legend(nrow = 8, title=" A. Phylum"))+
     ggtitle("B. Absolute Abundance of Lake Cadagno Bacterial Communites (Phylum)" )+
   theme(axis.text=element_text(size=14),
          axis.title=element_text(size=14,face="bold"), legend.text = element_text(size=12))+
        xlab("Depth (m)")+
    theme(
  # Hide panel borders and remove grid lines
  panel.grid.major = element_blank(),
  panel.grid.minor = element_blank(),
  # Change axis line 91A84C
  )+
  #theme(axis.title.y = element_blank(), axis.ticks.y = element_blank())+
    theme(plot.title = element_text(size = 16, face = "bold"))+ 
   # theme(legend.key.width=unit(1.5, "cm"))+

      annotate(geom="text", x=8, y=150000, label="Chemocline",
              color="black", size=5.5)+
      annotate(geom="text", x=14, y=150000, label="Mixolimnion",
              color="black", size=5.5) + 
      annotate(geom="text", x=4, y=152000, label="Monimolimnion",
              color="black", size=5.5)+
   theme(legend.position = c(0.55, 0.75), legend.background = element_rect(size=0.3, 
                                           colour ="white"),  legend.direction = "vertical")+
  theme(legend.title=element_text(size=12, face="bold"), 
    legend.text=element_text(size=12), axis.ticks.y = element_blank())+    scale_y_continuous(labels = scales::scientific, breaks = scales::pretty_breaks(n = 6), limits = c(0, 9e+05))+
         theme (legend.box.background = element_rect(colour = "black"))+
    theme(axis.title.y = element_blank(), axis.ticks.y = element_blank(), axis.text.y = element_blank())+
   theme(legend.position = "none")
 

  
 

 #Fig 3(B)
 
  jascolor1a<-    c("steelblue","#56B4E9", "lightblue1","lightcyan2","burlywood1","burlywood3","burlywood4", "plum1","plum3","plum4", "snow2", "snow3", "snow4", "dimgray", "palevioletred2", "hotpink1", "hotpink3", "tan2", "tan3", "tan4", "tomato1", "tomato3","tomato4","chartreuse3", "chartreuse4", "seagreen1","seagreen3","seagreen4")
                 

   Phylum_proteobacteria<- AA_Genus2[which(AA_Genus2$Phylum=="Proteobacteria"), ]
#  Phylum_proteobacteria<- Phylum_proteobacteria[which(Phylum_proteobacteria$AA_Arch_Bac>=1000), ]
 
   ProtOTU2_P <-        ggplot(data = Phylum_proteobacteria,
                   mapping = aes_string(x ="Sample",y="AA_Arch_Bac",fill="Genus"))+
                   geom_bar(stat="identity")+
                   theme(legend.position="right")+
                   #geom_text(aes(label=OTU),position=position_stack(0.5), angle = -90)+
                   # geom_label_repel(aes(label=OTU), position=position_stack(0.5))+
                   ylab("Absolute Abundance (>1000 Cells/mL)")+
                                     coord_flip()+                   theme_bw()+
                   scale_fill_manual(values = c(jascolor1a))+
                   xlab("Depth (m)")+
                   scale_x_discrete(labels=c("LC-19m-E1"="19","LC-17m-E1"="17",
                              "LC-15point5m-E1"="15.5", 
                              "Cadagno15m-E2"="15", "LC-13m-E1"="13", "LC-11m-E1"="11","LC-9m-E1"="9",
                              "Cadagno7m-E2"="7", "LC-5m-E1"="5", "LC-3m-E1"="3"), 
                      limits=c("", "LC-19m-E1","LC-17m-E1"," ","LC-15point5m-E1", "Cadagno15m-E2",
                              "LC-13m-E1"," ", "LC-11m-E1", "LC-9m-E1", "Cadagno7m-E2",
                              "LC-5m-E1", "LC-3m-E1", "")) +  
                   theme(legend.position = "left")+
                   theme(axis.text = element_text(size = 12))+
                   theme(axis.title = element_text(size = 12),  legend.box.background = element_rect(color="white",                    size=01), panel.border = element_rect(colour = "black", fill=NA, size=2))+
                   guides(fill=guide_legend(nrow=14, title.position = "top"))+
                   ggtitle("C. Absolute Abundance of Proteobacteria (Genus)" )+
                   theme(legend.position="right")+
                   theme(axis.text=element_text(size=14),
                   axis.title=element_text(size=14,face="bold"))+
                   xlab("")+
                   theme(
                  # Hide panel borders and remove grid lines
                   panel.grid.major = element_blank(),
                   panel.grid.minor = element_blank(),
                   # Change axis line
                        )+
                   theme(axis.ticks.y = element_blank())+
              
  theme(plot.title = element_text(size = 16, face = "bold"))+
  theme(legend.position = c(0.53, 0.75), legend.direction = "vertical",
                   legend.background = element_rect(size=0.3, 
                                   colour ="transparent", fill = "transparent"), legend.text = element_text(color = "black"))+
guides(fill=guide_legend(nrow=14, title= "B. Genus"))+
  theme(legend.title=element_text(size=12, face = "bold"), legend.text=element_text(size=12))+
        scale_y_continuous(labels = scales::scientific)+
     theme (legend.box.background = element_rect(colour = "black"))+
     xlab("Depth (m)")+
     scale_y_continuous(labels = scales::scientific, breaks = scales::pretty_breaks(n = 6), limits = c(0, 9e+05))
   
   
   
   
   #Fig 3(C)
   
    Phylum_Bacteroidetes<- AA_Genus2[which(AA_Genus2$Phylum=="Bacteroidetes"), ]
  #  Phylum_Bacteroidetes<- Phylum_Bacteroidetes[which(Phylum_Bacteroidetes$AA_Arch_Bac>=1000), ]
    
      jascolor1ab<-    c("steelblue","#56B4E9", "lightblue1","lightcyan2","burlywood1","burlywood3","burlywood4", "plum1","plum3","plum4", "snow2", "snow3", "snow4", "dimgray", "palevioletred2", "hotpink1", "hotpink3", "tan2", "tan3", "tan4", "tomato1", "tomato3","tomato4", "seagreen1","seagreen3","seagreen4")

    ProtOTU2_B <-        ggplot(data = Phylum_Bacteroidetes,
                   mapping = aes_string(x ="Sample",y="AA_Arch_Bac",fill="Genus"))+
                   geom_bar(stat="identity")+
                   theme(legend.position="right")+
                   #geom_text(aes(label=OTU),position=position_stack(0.5), angle = -90)+
                   # geom_label_repel(aes(label=OTU), position=position_stack(0.5))+
                   ylab("Absolute Abundance (>1000 Cells/mL)")+
                                     coord_flip()+                   theme_bw()+
                   scale_fill_manual(values = c(jascolor1ab))+
                   xlab("Depth (m)")+
                   scale_x_discrete(labels=c("LC-19m-E1"="19m","LC-17m-E1"="17m",
                              "LC-15point5m-E1"="15.5m", 
                              "Cadagno15m-E2"="15m", "LC-13m-E1"="13m", "LC-11m-E1"="11m","LC-9m-E1"="9m",
                              "Cadagno7m-E2"="7m", "LC-5m-E1"="5m", "LC-3m-E1"="3m"), 
                      limits=c("", "LC-19m-E1","LC-17m-E1"," ","LC-15point5m-E1", "Cadagno15m-E2",
                              "LC-13m-E1"," ", "LC-11m-E1", "LC-9m-E1", "Cadagno7m-E2",
                              "LC-5m-E1", "LC-3m-E1", "")) +  
                   theme(legend.position = "left")+
                   theme(axis.text = element_text(size = 12))+
                   theme(axis.title = element_text(size = 12),  legend.box.background = element_rect(color="white",                    size=01), panel.border = element_rect(colour = "black", fill=NA, size=2))+
                   guides(fill=guide_legend(ncol=2, title.position = "top"))+
                   ggtitle("D. Absolute Abundance of Bacteroidetes (Genus)" )+
                   theme(legend.position="right")+
                   theme(axis.text=element_text(size=14),
                   axis.title=element_text(size=14,face="bold"))+
                   xlab("")+
                   theme(
                  # Hide panel borders and remove grid lines
                   panel.grid.major = element_blank(),
                   panel.grid.minor = element_blank(),
                   # Change axis line
                        )+
                   theme(axis.title.y = element_blank(), axis.ticks.y = element_blank(), axis.text.y =                                 element_blank())+
              
  theme(plot.title = element_text(size = 16, face = "bold"))+
  theme(legend.position = c(0.63, 0.75), legend.direction = "vertical",
                   legend.background = element_rect(size=0.3, 
                                   colour ="transparent", fill = "transparent"), legend.text = element_text(color = "black"))+
guides(fill=guide_legend(nrow=13, title="C. Genus" ))+     
  theme(legend.title=element_text(size=12, face = "bold"), legend.text=element_text(size=12))+
              scale_y_continuous(labels = scales::scientific)+
           theme (legend.box.background = element_rect(colour = "black"))+
           scale_y_continuous(labels = scales::scientific, breaks = scales::pretty_breaks(n = 6), limits = c(0, 9e+05))

    
    
  
   
    
    
    #15 x15.5
     
  
  
  #16.8x16.8
  figure3
  
  #generate 39 color palette
  n <- 20
palette39 <- distinctColorPalette(n)

 

```

```{r fig.4, echo=TRUE}

#Flow-cytometry Cyanobacteria-like 
cyano_cnt_R <- read.csv("~/GitHub/Lake_Cadagno_microbial_loop_Saini_et_al_2021/Fig.1-4_Suppl/PLP_VLP_R.csv") 

#16s Cyanobacteria Absolute Abundance
 uahbs234ab_cyna_CPF <- read.csv("~/GitHub/Lake_Cadagno_microbial_loop_Saini_et_al_2021/16s_Files_R_codes/SI_Table3.csv")
 


  

 
#Fig 4 (A)
 p_c<- ggplot(cyano_cnt_R, aes(x=Depth, y=Cyn_Cells_mean)) + 
    geom_point(size=3.5)+
    geom_errorbar(aes(ymin=Cyn_Cells_mean-Cyn_Cells_sd, ymax=Cyn_Cells_mean+Cyn_Cells_sd), width=.2,
                  position=position_dodge(0.05))+ coord_flip()+
    ylab("Flow-cytometry Cyanobacteria-like (Cells/mL)")+ 
    xlab("Depth (m)")+
    theme(axis.text=element_text(size=24),
          axis.title=element_text(size=24))+
  coord_flip() + theme_bw()+
    annotate("rect", ymin=-Inf, ymax=Inf, xmin=-Inf, xmax=12.5, alpha=0.1, fill="green4")+
    annotate("rect", ymin=-Inf, ymax=Inf, xmin=12.5, xmax=13.5, alpha=0.2, fill="deeppink4")+
    annotate("rect", ymin=-Inf, ymax=Inf, xmin=13.5, xmax=15.5, alpha=0.5, fill="deeppink4")+
    annotate("rect", ymin=-Inf, ymax=Inf, xmin=15.5, xmax=16.5, alpha=0.2, fill="deeppink4")+
    annotate("rect", ymin=-Inf, ymax=Inf, xmin=16.5, xmax=Inf, alpha=0.2, fill="pink4")+
    theme(axis.text=element_text(size=24),axis.title=element_text(size=24))+
    theme(axis.line = element_line(color = "black"))+
    scale_x_reverse(limits = c(21,1), breaks=c(17, 15, 15.5,14, 13, 11,9, 7,5, 3, 1))+
    theme(legend.position = "none")+
     theme(axis.text=element_text(size=24),
          axis.title=element_text(size=24))+
    theme(
  # Hide panel borders and remove grid lines
  panel.grid.major = element_blank(),
  panel.grid.minor = element_blank(),
  # Change axis line
  )+
  
    # theme(axis.title.y = element_blank(), axis.text.y = element_blank(), axis.ticks.y= element_blank())+
      geom_segment(aes(x = 1 , y = 23666, xend = 2, yend = 25333), linetype="dashed", color = "black", size=2)+
    geom_segment(aes(x = 2 , y = 25333, xend = 3, yend = 21666), linetype="dashed", color = "black", size=2)+
    geom_segment(aes(x = 3 , y = 21666, xend = 4, yend = 32333), linetype="dashed", color = "black", size=2)+
    geom_segment(aes(x = 4 , y = 32333, xend = 5, yend =  41666), linetype="dashed", color = "black", size=2)+
    geom_segment(aes(x = 5 , y = 41666, xend = 6, yend = 52333), linetype="dashed", color = "black", size=2)+
    geom_segment(aes(x = 6 , y = 52333, xend = 7, yend = 42333), linetype="dashed", color = "black", size=2)+
    geom_segment(aes(x = 7 , y = 42333, xend = 8, yend = 30666), linetype="dashed", color = "black", size=2)+
    geom_segment(aes(x = 8 , y = 30666, xend = 9, yend = 43666), linetype="dashed", color = "black", size=2)+
    geom_segment(aes(x = 9 , y = 43666, xend =10, yend = 43333), linetype="dashed", color = "black", size=2)+
    geom_segment(aes(x = 10 , y = 43333, xend = 11, yend =  25333), linetype="dashed", color = "black", size=2)+
    geom_segment(aes(x = 11 , y =  25333, xend = 12, yend =  40333), linetype="dashed", color = "black", size=2)+
    geom_segment(aes(x = 12 , y =  40333, xend = 13, yend = 29000), linetype="solid", color = "black", size=2)+
    geom_segment(aes(x = 13 , y = 29000, xend = 13.5, yend = 24500), linetype="dashed", color = "black", size=2)+
    geom_segment(aes(x = 13.5 , y = 24500, xend = 14, yend = 309333), linetype="solid", color = "black", size=2)+
    geom_segment(aes(x = 14 , y = 309333, xend = 15, yend = 212333), linetype="solid", color = "black", size=2)+
    geom_segment(aes(x = 15 , y = 212333, xend = 15.5, yend = 190000), linetype="solid", color = "black", size=2)+
        geom_segment(aes(x = 15.5 , y = 190000, xend = 16, yend = 171666), linetype="solid", color = "black", size=2)+
        geom_segment(aes(x = 16 , y = 171666, xend = 17, yend = 166666), linetype="solid", color = "black", size=2)+
        geom_segment(aes(x = 17 , y = 166666, xend = 18, yend = 169333), linetype="solid", color = "black", size=2)+
        geom_segment(aes(x = 18 , y = 169333, xend = 19, yend = 148000), linetype="solid", color = "black", size=2)+
  # scale_y_continuous(labels = scales::scientific
#, sec.axis = sec_axis(~./12500, name = "Phycocyanin (μg/L)"))+
 #       theme(axis.title.x.top = element_text(color = "lightsalmon4"))+
  annotate("text", x = 2, y = 2e+05, label = "(- - -) 28 August 2017", colour = "black", size = 8.5)+   
  annotate("text", x = 3, y = 2e+05, label = "( — ) 29 August 2017", colour = "black", size = 8.5)+
          theme(panel.border = element_rect(colour = "black", fill=NA, size=2))+
  expand_limits(x = 0, y = 0)+
   ylab("Concentration (Cells/ml)")+
   ggtitle("C. Phycobilisome Containing Cells")+
    theme(plot.title = element_text(size = 26, face = "bold"))
   


#16s cyanobacteria AA uahbs234ab_cyna_CPF
#Fig 4 (B)
     uahbs234ab_cyna_CPF2<- uahbs234ab_cyna_CPF[which(uahbs234ab_cyna_CPF$AA_Arch_Bac>=100), ]


 Phylum_cyano2abn_CC <-        ggplot(data = uahbs234ab_cyna_CPF2,
                            mapping = aes_string(x ="Sample",y="AA_Arch_Bac",fill="Genus"))+
    geom_bar(stat="identity")+
    theme(legend.position="right")+
     theme(legend.text=element_text(size=24))+
    ylab("Absolute Abundance Phylum (>100 Cells/mL)")+
    coord_flip()+                   theme_bw()+
    xlab("Depth (m)")+
    scale_fill_manual(values=c( "darkolivegreen3","darkred","darkblue", "#52854C", "orange"))+
    scale_x_discrete(labels=c("LC-19m-E1"="19","LC-17m-E1"="17",
                              "LC-15point5m-E1"="15.5", 
                              "Cadagno15m-E2"="15", "LC-13m-E1"="13", "LC-11m-E1"="11","LC-9m-E1"="9",
                              "Cadagno7m-E2"="7", "LC-5m-E1"="5", "LC-3m-E1"="3"), 
                      limits=c("", "LC-19m-E1","LC-17m-E1"," ","LC-15point5m-E1", "Cadagno15m-E2",
                              "LC-13m-E1"," ", "LC-11m-E1", "LC-9m-E1", "Cadagno7m-E2",
                              "LC-5m-E1", "LC-3m-E1", "")) +  
    theme(legend.position = "left")+
    theme(legend.box.background = element_rect(color="white"), legend.direction = "vertical", panel.border = element_rect(colour = "black", fill=NA, size=2))+
     guides(fill=guide_legend(ncol=1, title="Genus"))+
    # ggtitle("Absolute Abundance Cyanobacteria (including Chloroplast)" )+
   theme(axis.text=element_text(size=24),
          axis.title=element_text(size=24))+
       xlab("")+
    theme(
  # Hide panel borders and remove grid lines
  panel.grid.major = element_blank(),
  panel.grid.minor = element_blank(),
  # Change axis line
  )+
  theme(axis.title.y = element_blank(), axis.ticks.y = element_blank())+
  scale_y_continuous(labels = function(x) format(x, scientific = TRUE))


    yv1abz_Phylum_cyano2abn_CC<- Phylum_cyano2abn_CC + 
      annotate(geom="text", x=8, y=10000, label="Chemocline",
              color="black", size=8.5)+
      annotate(geom="text", x=14, y=10000, label="Mixolimnion",
              color="black", size=8.5) + 
      annotate(geom="text", x=4, y=10000, label="Monimolimnion",
              color="black", size=8.5)+
   theme(axis.title.y = element_blank())+
   # theme(axis.text.y = element_blank())+
    theme(axis.ticks.y = element_blank())+
  theme(legend.title=element_text(size=24),
    legend.text=element_text(size=24)) +
      scale_y_continuous(labels = scales::scientific)+
    theme(legend.position = c(0.65, 0.80), legend.background = element_rect(size=0.3, 
                                           colour ="black"),  legend.direction = "vertical")+
      theme(legend.title=element_text(size=26), 
    legend.text=element_text(size=24))+
     scale_y_continuous(labels = scales::scientific)+
         theme(legend.key.width=unit(1.5, "cm"))+
      theme(legend.key.size = unit(2.5, 'lines'))+
         ylab("Concentration (>100 Cells/ml)")+
                theme(axis.title=element_text(size=24))+
       ggtitle("A. Cyanobacteria-classified Genera")+
    theme(plot.title = element_text(size = 26, face = "bold"))


    
    
   ##Assign category to Chloroplast and non chloroplast OTUs
    
#cyano_Cells<-    uahbs234ab_cyna_CPF2 %>%
#  mutate(Chl_cat = case_when(
#    endsWith(Genus, "Chloroplast_ge") ~ "Chloroplast",
#    endsWith(Genus, "Oxyphotobacteria_unclassified") ~ "Chloroplast", endsWith(Genus, "Gastranaerophilales_ge") ~ "Non-chloroplast",
#    endsWith(Genus, "Cyanobium_PCC-6307") ~ "Non-chloroplast",
#    endsWith(Genus, "Pseudanabaena_PCC-7429") ~ "Non-chloroplast"))


 cyano_Cells2 = dplyr::mutate(uahbs234ab_cyna_CPF2, TAX = paste(OTU, Genus,  sep='|'))
 
 

 
 
 cyano_Cells2$TAX <- factor(cyano_Cells2$TAX, levels = c("Otu00052|Chloroplast_ge", "Otu00042|Chloroplast_ge", "Otu00051|Chloroplast_ge", "Otu00033|Chloroplast_ge","Otu00055|Chloroplast_ge", "Otu00143|Chloroplast_ge",               "Otu00208|Chloroplast_ge","Otu00006|Chloroplast_ge", "Otu00323|Oxyphotobacteria_unclassified", "Otu00342|Oxyphotobacteria_unclassified", "Otu00008|Oxyphotobacteria_unclassified", "Otu00475|Gastranaerophilales_ge", "Otu00106|Cyanobium_PCC-6307", "Otu00286|Pseudanabaena_PCC-7429"))
 
 
 
     newOTU1BN <-        ggplot(data = cyano_Cells2,
                            mapping = aes_string(x ="Sample",y="AA_Arch_Bac",fill="TAX"))+
    theme(legend.position="right")+
        geom_bar(stat="identity")+
     ylab("Absolute Abundance of Chloroplast (>100 Cells/ml)")+
    theme(axis.text = element_text(size = 14))+
    theme(axis.title = element_text(size = 14))+
    coord_flip()+                   theme_bw()+
     scale_fill_manual(values = c("brown2", "brown3", "brown4", "deepskyblue4", "deepskyblue3", "darkblue", "darkorchid4", "chartreuse4", "chartreuse3", "chartreuse1", "yellowgreen", "darkgoldenrod2", "darkgoldenrod3", "darkgoldenrod4"))+
    xlab("Depth (m)")+
    theme(legend.position = "left")+
      theme(axis.text = element_text(size = 14))+
    theme(axis.title = element_text(size = 14),  legend.box.background = element_rect(color="black", size=01), panel.border = element_rect(colour = "black", fill=NA, size=2))+
     guides(fill=guide_legend(ncol=2, title="OTU|Genus", legend.position = "top"))+
     #ggtitle("Absolute Abundance of Chloroplast" )+
       theme(legend.position="right")+
   theme(axis.text=element_text(size=24),
          axis.title=element_text(size=24))+
       xlab("Depth (m)")+
    theme(
  # Hide panel borders and remove grid lines
  panel.grid.major = element_blank(),
  panel.grid.minor = element_blank(),
  # Change axis line
  )+
    scale_x_discrete(labels=c("LC-19m-E1"="19","LC-17m-E1"="17",
                              "LC-15point5m-E1"="15.5", 
                              "Cadagno15m-E2"="15", "LC-13m-E1"="13", "LC-11m-E1"="11","LC-9m-E1"="9",
                              "Cadagno7m-E2"="7", "LC-5m-E1"="5", "LC-3m-E1"="3"), 
                      limits=c("", "LC-19m-E1","LC-17m-E1"," ","LC-15point5m-E1", "Cadagno15m-E2",
                              "LC-13m-E1"," ", "LC-11m-E1", "LC-9m-E1", "Cadagno7m-E2",
                              "LC-5m-E1", "LC-3m-E1", ""))+
    annotate(geom="text", x=8, y=20000, label="Chemocline",
              color="black", size=8.5)+
      annotate(geom="text", x=14, y=20000, label="Mixolimnion",
              color="black", size=8.5) + 
      annotate(geom="text", x=4, y=20000, label="Monimolimnion",
              color="black", size=8.5)+
    theme(axis.ticks.y = element_blank())+
      scale_y_continuous(labels = scales::scientific)+
     theme(legend.position = c(0.65, 0.75))+
     theme(legend.title=element_text(size=26), 
    legend.text=element_text(size=24))+
    theme(legend.key.width=unit(1.5, "cm"))+
    theme(legend.key.size = unit(2.5, 'lines'))+
           theme(axis.text.y = element_blank(), axis.title.y = element_blank())+
           ylab("Concentration (>100 Cells/ml)")+
                theme(axis.title=element_text(size=24))+
        ggtitle("B. Putative Chloroplast and  Non-Chloroplast OTUs")+
    theme(plot.title = element_text(size = 26, face = "bold"))+
       guides(fill=guide_legend(nrow=7))
     
     
        
         

    
    
    
    #Figure 4 (E)   Phylogenetic Tree Annotation
     Meta_Sheet1 <- read.csv("~/GitHub/Lake_Cadagno_microbial_loop_Saini_et_al_2021/Fig.1-4_Suppl/rxaml_2_anc_tree - Sheet1.csv")
     Meta_Sheet2 <- read.csv("~/GitHub/Lake_Cadagno_microbial_loop_Saini_et_al_2021/Fig.1-4_Suppl/metafile_tree.csv")
     meta_sheet12<- merge(Meta_Sheet2, Meta_Sheet1, by="Accession_ID", all.x = F)

   Accession_ID<- c("AB294340.1.1396","AB930535.1.1412", "AY948033.1.1412","AY948048.1.1412",  "DQ422812.109483.110989", "DQ422812.150364.151870", "EF520516.1.1247", "EU580489.1.1445",
"FJ381978.1.1348" , "FJ381993.1.1344",  "FJ612150.1.1429",  "FJ612356.1.1446",
"FJ612436.1.1451",  "FJ968741.89400.90990" ,  "GU936924.1.1458", "JF272031.1.1446",
"JF495289.1.1445",  "JF917234.1.1452",  "JN626489.1.1342",  "JX221840.1.1399",
"KJ201907.99785.101255",  "KT199255.49748.51262", "LC016779.1.1996",  "LC153790.1.2050",
"Y17632.1.2184","CP003940.1433867.1435344", "CP003940.1167849.1169324",   "Otu00006", "Otu00008" ,"Otu00033", "Otu00042", "Otu00051", "Otu00052", "Otu00055", "Otu00106", "Otu00143",  "Otu00208", "Otu00286", "Otu00323" , "Otu00342", "Otu00475")
   
   OTU<- c("Otu00006", "Otu00008" ,"Otu00033", "Otu00042", "Otu00051", "Otu00052", "Otu00055", "Otu00106", "Otu00143",  "Otu00208", "Otu00286", "Otu00323" , "Otu00342", "Otu00475")
 OTUdf<- data.frame(OTU)
   
   OtudfT <- data.frame(Accession_ID)
   withotudf<- merge(meta_sheet12, OtudfT, by= "Accession_ID", all=T)
   

     OTU_ID_cya<- uahbs234ab_cyna_CPF %>%                           #Cyanobacteria AA 16s
     dplyr::  select("Sample", "Genus", "AA_Arch_Bac", "OTU")
   
    withotudf2<- merge(withotudf, OTU_ID_cya, by.x= "Accession_ID",by.y="OTU",all=T)
    
    withotudf3 = dplyr::mutate(withotudf2, newlab = paste(Accession_ID, Microbe, sep='| '))
    withotudf4 = dplyr::mutate(withotudf3, newlab2 = paste(newlab, Source, sep=', '))

withotudf4[] <- lapply(withotudf4, sub, pattern = "\\| NA.*", replacement = "")
        withotudf4<- withotudf4[,-c(2:4, 11:13, 20,21,23,25, 26,32,35, 37,38,43)]
withotudf56 <- withotudf4[-c(5,7,9,11), ]

 treenex1_O<- read.tree("C:/Users/sooch/Desktop/RAxML_bipartitions.T23")
 treenex1_O$tip.label<- gsub(".*Otu","Otu", treenex1_O$tip.label)
treenex1_O$tip.label<- gsub("\\_.*","",treenex1_O$tip.label)


 treem2 <- ggtree(treenex1_O, size=1, linetype="solid") +
   geom_text2(aes(label=node))+
 
           geom_text2(aes(label=label, subset = !is.na(as.numeric(label)) & as.numeric(label) > 80), color="black") 

    
    lbc5A <- groupClade(treenex1_O, c(47,56,60,62,44))
    
   

lbc6A<- ggtree(lbc5A, aes(color=group), size=3)+ 
       scale_color_manual(values=c("black", "brown4", "deepskyblue3", "darkblue",  "darkgoldenrod3", "darkorchid4"))+
       #geom_text2(aes(label=label, subset = !is.na(as.numeric(label)) & as.numeric(label) > 80), color="black")+
  geom_label2 (aes(label=label, subset = !is.na(as.numeric(label)) & as.numeric(label) > 80), color="black", size=8, hjust=.5)
#+geom_text(aes(label=node))


lbc7A<- lbc6A %<+% withotudf56




lbc8A <- lbc7A+   
  geom_treescale(linesize=4, fontsize = 9)+
  geom_tiplab(aes(label=newlab2), size=9) +
  geom_tippoint(aes(shape=Genus), size=9)+
    scale_shape_manual( "SILVA Genus", values=c(19,18,15,17,8), labels = c("Chloroplast", "Cyanobium_PCC-6307", "Gastranaerophilales","Oxyphotobacteria_unclassified", "Pseudanabaena_PCC-7429", ""))+

   guides(color = FALSE, fill= FALSE)+
  theme(legend.background = element_rect(size=2, linetype="solid"))+
  theme(legend.text=element_text(size=21), legend.title=element_text(size=22, face="bold"), legend.position = c(0.85, 0.90), legend.background = element_rect(size=0.3, 
                                           colour ="black"),  legend.direction = "vertical")+
   ggtitle("Phylogram with manual bootstrapping (Values <80)")+
    theme(plot.title = element_text(hjust = 0.5, size=22, face = "bold"))+
  theme(plot.margin = unit(c(0, 0, 0, 0), "cm"))+
  xlim(0,1)+

  geom_cladelabel(node=47, label="Uncultured Chloroplast", color="brown4", offset=0.2,
                           align=T, angle=270, hjust='center', barsize = 3,
  fontsize = 10.5)+


   geom_cladelabel(node=56, color="deepskyblue3", label="Eukaryota; Sar; Stramenopiles; Ochrophyta",
                  offset=.3, align=T, hjust='center', angle=270, barsize = 3,
  fontsize = 10.5 )+

   geom_cladelabel(node=60, label="Eukaryota; Haptista; Haptophyta", color="darkblue", 
                    offset=.2, align=T, angle=270, hjust='center', barsize = 3,
  fontsize = 10.5 )+
   geom_cladelabel(node=62, label="Cyanobacteria", color= "darkgoldenrod3", 
                        offset=.35, align=T, angle=270, hjust='center', barsize = 3,
  fontsize = 10.5 )+


   geom_cladelabel(node=74, label="Eukaryota; Viridiplantae; Chlorophyta", color= "yellowgreen", 
                        offset=.2, align=T, angle=270, hjust='center', barsize = 3,
  fontsize = 10.5 )+
 geom_cladelabel(node=44, label="Eukaryota; Viridiplantae; Streptophyta", color="darkorchid4", 
                      offset=.3, align=T, angle=270, hjust='center', barsize = 3,
  fontsize = 10.5 )

Figure4<- ggarrange(yv1abz_Phylum_cyano2abn_CC, newOTU1BN, p_c, ncol=3, widths = c(0.6,1,0.35), align = "h")


#Figure4<- ggarrange(p_c, yv1abz_Phylum_cyano2abn_CC, newOTU1BN, newOTU2, ncol=4, widths = c(0.6,1,0.8,0.8), align = "h")
 
 Figure4 / plot_spacer()/ lbc8A + plot_layout(heights = c(1,0.05, 1.75)) #32x36
    

```

```{r fig.S5,Fig.S6, echo=TRUE}

#fig.S2(B)

names(FvFm_Ind)[1] <- "Depth_m"
names(FvFm_Ind)[2] <- "Fb"
names(FvFm_Ind)[3] <- "Fm"
names(FvFm_Ind)[4] <- "Fr"



df2 <- mutate_all(FvFm_Ind, function(x) as.numeric(as.character(x)))
df3<- df2 %>%
      group_by(Depth_m)  %>%
    dplyr::summarize(mean_Fb = mean(Fb, na.rm = TRUE), SD_fb = sd(Fb, na.rm = TRUE), mean_Fm = mean(Fm, na.rm = TRUE), SD_fm = sd(Fm, na.rm = TRUE), mean_Fr= mean(Fr, na.rm = TRUE), SD_fr= sd(Fr, na.rm = TRUE))
df3B <- mutate_all(df3, function(x) as.numeric(as.character(x)))

df3BC<- df3B[!is.na(df3B$mean_Fr & df3B$SD_fr),]

#Fig S2 (B)
dt<- ggplot(data=df3B, aes(y=mean_Fb, x=Depth_m))+ 
  geom_point(data=df3B, aes(y=mean_Fb, x=Depth_m, color="mean_Fb"))+
  geom_line(data=df3B, aes(y=mean_Fb, x=Depth_m, color="mean_Fb"))+
  geom_errorbar(data=df3B, aes(ymin=as.numeric(df3B$mean_Fb-df3B$SD_fb), ymax=as.numeric(mean_Fb+df3B$SD_fb), color="mean_Fb", na.rm = TRUE), width=.2,position=position_dodge(0.01))+
  geom_point(data=df3B, aes(y=mean_Fm, x=Depth_m, color="mean_Fm"))+
  geom_line(data=df3B, aes(y=mean_Fm, x=Depth_m, color="mean_Fm"))+
  geom_errorbar(data=df3B, aes(ymin=as.numeric(df3B$mean_Fm-df3B$SD_fm), ymax=as.numeric(mean_Fm+df3B$SD_fm), color="mean_Fm"), width=.2,position=position_dodge(0.01))+
   geom_point(data=df3BC, aes(y=mean_Fr, x=Depth_m, color="mean_Fr"))+
  geom_line(data=df3BC, aes(y=mean_Fr, x=Depth_m, color="mean_Fr"))+
   scale_colour_manual("", breaks = c("mean_Fb", "mean_Fm", "mean_Fr"),
  values = c("blue", "black", "red"), labels=c("Basal fluorescence (Fb)", "Maximum fluorescence (Fm)", "Blank (Fr)"))+
  coord_flip()+ theme_bw()+
   scale_x_reverse(limits = c(21,1), breaks=c(19, 17, 15.5, 15, 14, 13, 11,9, 7,5, 3, 1))+
  xlab("Depth (m)")+
  ylab("fluorescence value")+
    annotate("rect", ymin=-Inf, ymax=Inf, xmin=-Inf, xmax=12.5, alpha=0.1, fill="green4")+
     annotate("rect", ymin=-Inf, ymax=Inf, xmin=12.5, xmax=13.5, alpha=0.2, fill="deeppink4")+
     annotate("rect", ymin=-Inf, ymax=Inf, xmin=13.5, xmax=15.5, alpha=0.5, fill="deeppink4")+
     annotate("rect", ymin=-Inf, ymax=Inf, xmin=15.5, xmax=16.5, alpha=0.2, fill="deeppink4")+
     annotate("rect", ymin=-Inf, ymax=Inf, xmin=16.5, xmax=Inf, alpha=0.2, fill="pink4")+
       theme(axis.text=element_text(size=12),axis.title=element_text(size=12,face="bold"))+
  theme(
  # Hide panel borders and remove grid lines
  panel.grid.major = element_blank(),
  panel.grid.minor = element_blank(),
  # Change axis line 
  )+theme(legend.position = c(0.55, 0.65))+
  theme(legend.background = element_rect(size=0.3, colour ="transparent", fill = "transparent"))
 
  
Meta_data_PLP_VLP_II <-  read.csv("~/GitHub/Lake_Cadagno_microbial_loop_Saini_et_al_2021/Fig.1-4_Suppl/RA_div_all3.csv")

#Fig.S3

Sp_chl<- ggplot(Meta_data_PLP_VLP_II, aes(y=Meta_data_PLP_VLP_II$CTD_28_29_chl, x=Meta_data_PLP_VLP_II$Bacterial_production)) +
    geom_point() +
  stat_smooth(method = "lm", color = "black", size=0.3)+
  stat_cor(method = "pearson", label.y = 20, label.x = 0.0000005)+ theme_bw()+
    xlab("Secondary Production (ngC L-1 H-1)")+
    ylab("Day 1 & Day 2 Average Chl a (µg/L)")+
     geom_point(aes(color=as.factor(Depth_m)), size=4)+
     scale_color_manual("Depth (m)", values=c("#52854C", "#C3D7A4","darkseagreen","forestgreen", "darkolivegreen3", "mediumorchid1", "deeppink4", "deeppink1", "#CC79A7", "peachpuff2", "peachpuff4"))


Sp_phy<- ggplot(Meta_data_PLP_VLP_II, aes(y=Meta_data_PLP_VLP_II$CTD2_Phycocyanin_29, x=Meta_data_PLP_VLP_II$Bacterial_production)) +
    geom_point() +
  stat_smooth(method = "lm", color = "black", size=0.3)+
  stat_cor(method = "pearson", label.y = 20, label.x = 0.0000005)+ theme_bw()+
    xlab("Secondary Production (ngC L-1 H-1)")+
    ylab("Day 2 Phycocyanin (µg/L)")+
     geom_point(aes(color=as.factor(Depth_m)), size=4)+
     scale_color_manual("Depth (m)", values=c("#52854C", "#C3D7A4","darkseagreen","forestgreen", "darkolivegreen3", "mediumorchid1", "deeppink4", "deeppink1", "#CC79A7", "peachpuff2", "peachpuff4"))


Sp_oxy<- ggplot(Meta_data_PLP_VLP_II, aes(y=Meta_data_PLP_VLP_II$CTD_28_29_Oxygen, x=Meta_data_PLP_VLP_II$Bacterial_production)) +
    geom_point() +
  stat_smooth(method = "lm", color = "black", size=0.3)+
  stat_cor(method = "pearson", label.y = 10, label.x = 0.0000005)+ theme_bw()+
    xlab("Secondary Production (ngC L-1 H-1)")+
    ylab("Day 1 & Day 2 Average Oxygen (mg/L)")+
     geom_point(aes(color=as.factor(Depth_m)), size=4)+
     scale_color_manual("Depth (m)", values=c("#52854C", "#C3D7A4","darkseagreen","forestgreen", "darkolivegreen3", "mediumorchid1", "deeppink4", "deeppink1", "#CC79A7", "peachpuff2", "peachpuff4"))



SP_par<- ggplot(Meta_data_PLP_VLP_II, aes(y=Meta_data_PLP_VLP_II$CTD2_PAR_29, x=Meta_data_PLP_VLP_II$Bacterial_production)) +
    geom_point() +
  stat_smooth(method = "lm", color = "black", size=0.3)+
  stat_cor(method = "pearson", label.y = 200, label.x = 0.0000005)+ theme_bw()+
    xlab("Secondary Production (ngC L-1 H-1)")+
    ylab("Day 2 PAR (w/m^2)")+
     geom_point(aes(color=as.factor(Depth_m)), size=4)+
     scale_color_manual("Depth (m)", values=c("#52854C", "#C3D7A4","darkseagreen","forestgreen", "darkolivegreen3", "mediumorchid1", "deeppink4", "deeppink1", "#CC79A7", "peachpuff2", "peachpuff4"))


Sp_bio<- ggplot(Meta_data_PLP_VLP_II, aes(y=Meta_data_PLP_VLP_II$CTD_28_29_Turb, x=Meta_data_PLP_VLP_II$Bacterial_production)) +
    geom_point() +
  stat_smooth(method = "lm", color = "black", size=0.3)+
  stat_cor(method = "pearson", label.y = 10, label.x = 0.0000005)+ theme_bw()+
    xlab("Secondary Production (ngC L-1 H-1)")+
    ylab("Day 1 & Day 2 Average Turbidity (FTU)")+
     geom_point(aes(color=as.factor(Depth_m)), size=4)+
     scale_color_manual("Depth (m)", values=c("#52854C", "#C3D7A4","darkseagreen","forestgreen", "darkolivegreen3", "mediumorchid1", "deeppink4", "deeppink1", "#CC79A7", "peachpuff2", "peachpuff4"))

Sp_cells<- ggplot(Meta_data_PLP_VLP_II, aes(y=Meta_data_PLP_VLP_II$TCell_cnt_ml, x=Meta_data_PLP_VLP_II$Bacterial_production)) +
    geom_point() +
  stat_smooth(method = "lm", color = "black", size=0.3)+
  stat_cor(method = "pearson", label.y = 900000, label.x = 0.0000005)+ theme_bw()+
    xlab("Secondary Production (ngC L-1 H-1)")+
    ylab("PLPs (Cells/ml)")+
     geom_point(aes(color=as.factor(Depth_m)), size=4)+
     scale_color_manual("Depth (m)", values=c("#52854C", "#C3D7A4","darkseagreen","forestgreen", "darkolivegreen3", "mediumorchid1", "deeppink4", "deeppink1", "#CC79A7", "peachpuff2", "peachpuff4"))



chl_par<- ggplot(Meta_data_PLP_VLP_II, aes(y=Meta_data_PLP_VLP_II$CTD_28_29_chl, x=Meta_data_PLP_VLP_II$CTD2_PAR_29)) +
    geom_point() +
  stat_smooth(method = "lm", color = "black", size=0.3)+
  stat_cor(method = "pearson", label.y = 20, label.x = 50)+ theme_bw()+
    ylab("Day 1 & Day 2 Average Chl a (µg/L)")+
    xlab("Day 2 PAR (w/m^2)")+
     geom_point(aes(color=as.factor(Depth_m)), size=4)+
     scale_color_manual("Depth (m)", values=c("#52854C", "#C3D7A4","darkseagreen","forestgreen", "darkolivegreen3", "mediumorchid1", "deeppink4", "deeppink1", "#CC79A7", "peachpuff2", "peachpuff4"))

chl_oxygen<- ggplot(Meta_data_PLP_VLP_II, aes(y=Meta_data_PLP_VLP_II$CTD_28_29_chl, x=Meta_data_PLP_VLP_II$CTD_28_29_Oxygen)) +
    geom_point() +
  stat_smooth(method = "lm", color = "black", size=0.3)+
  stat_cor(method = "pearson", label.y = 20, label.x = 5)+ theme_bw()+
    ylab("Day 1 & Day 2 Average Chl a (µg/L)")+
    xlab("Day 1 & Day 2 Average Oxygen (mg/L)")+
     geom_point(aes(color=as.factor(Depth_m)), size=4)+
     scale_color_manual("Depth (m)", values=c("#52854C", "#C3D7A4","darkseagreen","forestgreen", "darkolivegreen3", "mediumorchid1", "deeppink4", "deeppink1", "#CC79A7", "peachpuff2", "peachpuff4"))


phy_par<- ggplot(Meta_data_PLP_VLP_II, aes(y=Meta_data_PLP_VLP_II$CTD2_Phycocyanin_29, x=Meta_data_PLP_VLP_II$CTD2_PAR_29)) +
    geom_point() +
  stat_smooth(method = "lm", color = "black", size=0.3)+
  stat_cor(method = "pearson", label.y = 20, label.x = 50)+ theme_bw()+
    ylab("Day 2 Phycocyanin (µg/L)")+
    xlab("Day 2 PAR (w/m^2)")+
     geom_point(aes(color=as.factor(Depth_m)), size=4)+
     scale_color_manual("Depth (m)", values=c("#52854C", "#C3D7A4","darkseagreen","forestgreen", "darkolivegreen3", "mediumorchid1", "deeppink4", "deeppink1", "#CC79A7", "peachpuff2", "peachpuff4"))

phy_oxygen<- ggplot(Meta_data_PLP_VLP_II, aes(y=Meta_data_PLP_VLP_II$CTD2_Phycocyanin_29, x=Meta_data_PLP_VLP_II$CTD_28_29_Oxygen)) +
    geom_point() +
  stat_smooth(method = "lm", color = "black", size=0.3)+
  stat_cor(method = "pearson", label.y = 20, label.x = 5)+ theme_bw()+
    ylab("Day 2 Phycocyanin (µg/L)")+
    xlab("Day 1 & Day 2 Average Oxygen (mg/L)")+
     geom_point(aes(color=as.factor(Depth_m)), size=4)+
     scale_color_manual("Depth (m)", values=c("#52854C", "#C3D7A4","darkseagreen","forestgreen", "darkolivegreen3", "mediumorchid1", "deeppink4", "deeppink1", "#CC79A7", "peachpuff2", "peachpuff4"))


##Figure S2

 Fig.S2F<-     ggarrange(Sp_chl, Sp_phy, Sp_bio, Sp_oxy, SP_par, Sp_cells, chl_par, chl_oxygen, phy_par, phy_oxygen, nrow = 5, ncol = 2, common.legend = T, legend= "bottom", labels = c("(A)", "(B)", "(C)"," (D)", "(E)", "(F)", "(G)", "(H)", "(I)", "(J)"), align = "v") #15x15 pdf
 
 
 
 
#fig.S3

 PLP_turb<- ggplot(Meta_data_PLP_VLP_II, aes(y=Meta_data_PLP_VLP_II$TCell_cnt_ml, x=Meta_data_PLP_VLP_II$CTD_28_29_Turb)) +
    geom_point() +
  stat_smooth(method = "lm", color = "black", size=0.3)+
  stat_cor(method = "pearson", label.y = 750000, label.x = 2)+ theme_bw()+
    xlab("Turbidity (FTU)")+
    ylab("PLPs/mL")+
     geom_point(aes(color=as.factor(Depth_m)), size=4)+
     scale_color_manual("Depth (m)", values=c("#52854C", "#C3D7A4","darkseagreen","forestgreen", "darkolivegreen3", "mediumorchid1", "deeppink4", "deeppink1", "#CC79A7", "peachpuff2", "peachpuff4"))
 
  PLP_chl<- ggplot(Meta_data_PLP_VLP_II, aes(y=Meta_data_PLP_VLP_II$TCell_cnt_ml, x=Meta_data_PLP_VLP_II$CTD_28_29_chl)) +
    geom_point() +
  stat_smooth(method = "lm", color = "black", size=0.3)+
  stat_cor(method = "pearson", label.y = 750000, label.x = 2)+ theme_bw()+
    ylab("PLPs/mL")+
    xlab("Chl (μg/L)")+
     geom_point(aes(color=as.factor(Depth_m)), size=4)+
     scale_color_manual("Depth (m)", values=c("#52854C", "#C3D7A4","darkseagreen","forestgreen", "darkolivegreen3", "mediumorchid1", "deeppink4", "deeppink1", "#CC79A7", "peachpuff2", "peachpuff4"))
  
   PLP_POC<- ggplot(Meta_data_PLP_VLP_II, aes(y=Meta_data_PLP_VLP_II$TCell_cnt_ml, x=Meta_data_PLP_VLP_II$POC)) +
    geom_point() +
  stat_smooth(method = "lm", color = "black", size=0.3)+
  stat_cor(method = "pearson", label.y = 750000, label.x = 1)+ theme_bw()+
    xlab("POC (ppm)")+
    ylab("PLPs/mL")+
     geom_point(aes(color=as.factor(Depth_m)), size=4)+
     scale_color_manual("Depth (m)", values=c("#52854C", "#C3D7A4","darkseagreen","forestgreen", "darkolivegreen3", "mediumorchid1", "deeppink4", "deeppink1", "#CC79A7", "peachpuff2", "peachpuff4"))
   
   
    PLP_POS<- ggplot(Meta_data_PLP_VLP_II, aes(y=Meta_data_PLP_VLP_II$TCell_cnt_ml, x=Meta_data_PLP_VLP_II$POS)) +
    geom_point() +
  stat_smooth(method = "lm", color = "black", size=0.3)+
  stat_cor(method = "pearson", label.y = 750000, label.x = 0.1)+ theme_bw()+
     xlab("POS (ppm)")+
    ylab("PLPs/mL")+
     geom_point(aes(color=as.factor(Depth_m)), size=4)+
     scale_color_manual("Depth (m)", values=c("#52854C", "#C3D7A4","darkseagreen","forestgreen", "darkolivegreen3", "mediumorchid1", "deeppink4", "deeppink1", "#CC79A7", "peachpuff2", "peachpuff4"))
    
     PLP_PON<- ggplot(Meta_data_PLP_VLP_II, aes(y=Meta_data_PLP_VLP_II$TCell_cnt_ml, x=Meta_data_PLP_VLP_II$PON)) +
    geom_point() +
  stat_smooth(method = "lm", color = "black", size=0.3)+
  stat_cor(method = "pearson", label.y = 100000, label.x = 0.25)+ theme_bw()+
      xlab("PON (ppm)")+
    ylab("PLPs/mL")+
     geom_point(aes(color=as.factor(Depth_m)), size=4)+
     scale_color_manual("Depth (m)", values=c("#52854C", "#C3D7A4","darkseagreen","forestgreen", "darkolivegreen3", "mediumorchid1", "deeppink4", "deeppink1", "#CC79A7", "peachpuff2", "peachpuff4"))
     
    


Fig.S3F<-     ggarrange(PLP_turb, PLP_chl, PLP_POC, PLP_POS, PLP_PON, NULL, nrow = 3, ncol = 2, common.legend = T, legend= "bottom", labels = c("(A)", "(B)", "(C)"," (D)", "(E)"))
Fig.S3F 


#Fig. S4

VLP_cor2 <- Meta_data_PLP_VLP_II[-c(2,10), ]

 VLP_oxy<- ggplot(VLP_cor2, aes(x=VLP_cor2$DO_mg_CTD1_29_cor, y=VLP_cor2$VLP_p_ml)) +
    geom_point() +
  stat_smooth(method = "lm", color = "black", size=0.3)+
 
  stat_cor(method = "pearson", label.y = 1.6e+08, label.x = 2.5)+ theme_bw()+
    xlab("Day 1 and Day 2 Oxygen (mg/L)")+
    ylab("VLP/mL")+
     geom_point(aes(color=as.factor(Depth_m)), size=4)+
     scale_color_manual("Depth (m)", values=c("darkseagreen","forestgreen", "darkolivegreen3", "mediumorchid1", "deeppink3", "deeppink4", "peachpuff2", "peachpuff4"))


 VLP_par<- ggplot(VLP_cor2, aes(x=VLP_cor2$CTD2_PAR_29, y=VLP_cor2$VLP_p_ml)) +
    geom_point() +
  stat_smooth(method = "lm", color = "black", size=0.3)+
 
  stat_cor(method = "pearson", label.y = 1.6e+08, label.x = 100)+ theme_bw()+
    xlab("Day 2 PAR (w/m^2)")+
    ylab("VLP/mL")+
     geom_point(aes(color=as.factor(Depth_m)), size=4)+
     scale_color_manual("Depth (m)", values=c("darkseagreen","forestgreen", "darkolivegreen3", "mediumorchid1", "deeppink3", "deeppink4", "peachpuff2", "peachpuff4"))
 
 
  VLP_dep<- ggplot(VLP_cor2, aes(x=VLP_cor2$Depth_m, y=VLP_cor2$VLP_p_ml)) +
    geom_point() +
  stat_smooth(method = "lm", color = "black", size=0.3)+
 
  stat_cor(method = "pearson", label.y = 1.6e+08, label.x = 3)+ theme_bw()+
    xlab("Depth (m)")+
    ylab("VLP/mL")+
     geom_point(aes(color=as.factor(Depth_m)), size=4)+
     scale_color_manual("Depth (m)", values=c("darkseagreen","forestgreen", "darkolivegreen3", "mediumorchid1", "deeppink3", "deeppink4", "peachpuff2", "peachpuff4"))
  
  
    VLP_cond<- ggplot(VLP_cor2, aes(x=VLP_cor2$CTD_28_29_Cond, y=VLP_cor2$VLP_p_ml)) +
    geom_point() +
  stat_smooth(method = "lm", color = "black", size=0.3)+
 
  stat_cor(method = "pearson", label.y = 1.6e+08, label.x = 0.15)+ theme_bw()+
    xlab("Day 1 and Day 2 Conductivity (mS/cm)") +
    ylab("VLP/mL")+
     geom_point(aes(color=as.factor(Depth_m)), size=4)+
     scale_color_manual("Depth (m)", values=c("darkseagreen","forestgreen", "darkolivegreen3", "mediumorchid1", "deeppink3", "deeppink4", "peachpuff2", "peachpuff4"))
    
        VLP_SP<- ggplot(VLP_cor2, aes(x=VLP_cor2$Bacterial_production, y=VLP_cor2$VLP_p_ml)) +
    geom_point() +
  stat_smooth(method = "lm", color = "black", size=0.3)+
 
  stat_cor(method = "pearson", label.y = 2e+08, label.x = 5e-07)+ theme_bw()+
    xlab("Secondary production (ngC L-1 H-1)") +
    ylab("VLP/mL")+
     geom_point(aes(color=as.factor(Depth_m)), size=4)+
     scale_color_manual("Depth (m)", values=c("darkseagreen","forestgreen", "darkolivegreen3", "mediumorchid1", "deeppink3", "deeppink4", "peachpuff2", "peachpuff4"))
        
#Fig. S4        
Fig.S4<- ggarrange(VLP_oxy, VLP_par, VLP_dep, VLP_cond, VLP_SP, NULL, nrow = 3, ncol= 2, common.legend = T, legend= "bottom", labels = c("(A)", "(B)", "(C)"," (D)", "(E)", "(F)"))
        Fig.S4
        
        
        

        #Fig S5
 pieipe3_ABN <-        ggplot(data = AA_Genus,
                          mapping = aes_string(x=  "Sample", y="OTU_RA", fill="Phylum"))+ theme_bw()+
     geom_bar(stat="identity")+
    theme(legend.position="right")+ 
     scale_fill_manual(values = c("steelblue","#56B4E9", "lightblue1","burlywood1","burlywood3", "palevioletred2", "hotpink1", "hotpink3", "chartreuse3", "palegreen",   "snow2", "snow3", "snow4", "dimgray","tan3", "tan2", "tan", "thistle1", "thistle2", "plum1"))+
    theme(legend.box.background = element_rect(color="white", size=2), panel.border = element_rect(colour = "black", fill=NA, size=2))+
      theme(axis.text = element_text(size = 12))+
     guides(fill=guide_legend(ncol = 1, title="A-B Phylum"))+
     ggtitle("A. Relative Abundance of Lake Cadagno Bacterial Communites (Phylum)" )+
   theme(axis.text=element_text(size=14),
          axis.title=element_text(size=14,face="bold"), legend.text = element_text(size=12))+
        xlab("Depth (m)")+
    theme(
  # Hide panel borders and remove grid lines
  panel.grid.major = element_blank(),
  panel.grid.minor = element_blank(),
  # Change axis line 91A84C
  ) +
  #theme(axis.title.y = element_blank(), axis.ticks.y = element_blank())+
    theme(plot.title = element_text(size = 16, face = "bold"))+ 
   # theme(legend.key.width=unit(1.5, "cm"))+

  theme(legend.title=element_text(size=12, face="bold"), 
    legend.text=element_text(size=12), axis.ticks.y = element_blank())+
    theme(legend.position = "right")+
    coord_flip()+  scale_x_discrete(labels=c("LC-19m-E1"="19","LC-17m-E1"="17",
                              "LC-15point5m-E1"="15.5", 
                              "Cadagno15m-E2"="15", "LC-13m-E1"="13", "LC-11m-E1"="11","LC-9m-E1"="9",
                              "Cadagno7m-E2"="7", "LC-5m-E1"="5", "LC-3m-E1"="3"), 
                      limits=c("", "LC-19m-E1","LC-17m-E1"," ","LC-15point5m-E1", "Cadagno15m-E2",
                           "LC-13m-E1"," ", "LC-11m-E1", "LC-9m-E1", "Cadagno7m-E2",
                              "LC-5m-E1", "LC-3m-E1", ""))+
    ylab("Relative Abundance")+
     annotate(geom="text", x=8, y=0.50, label="Chemocline",
              color="black", size=5.5)+
      annotate(geom="text", x=14, y=0.50, label="Mixolimnion",
              color="black", size=5.5) + 
      annotate(geom="text", x=4, y=0.50, label="Monimolimnion",
              color="black", size=5.5)+
    theme(legend.position = c(0.63, 0.75), legend.direction = "vertical",
                   legend.background = element_rect(size=0.3, 
                                   colour ="transparent", fill = "transparent"), legend.text = element_text(color = "black"))+
   guides(fill=guide_legend(nrow=20))
    
 #fig.3  (15x16)
  figure3<-   (pieipe3_ABN | yv1a_CC   +  plot_layout(guides = "collect")) / (ProtOTU2_P | ProtOTU2_B)
 
 
 
 palette1 <- distinctColorPalette(33)

    Relative_Genus<- AA_Genus2[which(AA_Genus2$AA_Arch_Bac>=5000), ]

    pieipe3_ABN_Genus <-        ggplot(data = Relative_Genus,
                          mapping = aes_string(x=  "Sample", y="AA_Arch_Bac", fill="TAX"))+ theme_bw()+
     geom_bar(stat="identity")+
    theme(legend.position="right")+ 
     scale_fill_manual (values = c("#9E9654", "#6DE5E2", "#7D589D",  "#BAE498", "#E486DB", "#71DE7A", "#E9D6DD", "#822EEC", "#BAE0E8", "#E44AC0", "#C7E7CB", "#6DE8B9", "#7342D5","#E6DD6D", "#7DB2A0", "#D6478B","#D4E741", "#E2DDAB", "#68C5E1", "#E15F4E", "#E98AA7", "#A96E8B", "#B7B9DA", "#DFAFDC", "#66A2E4", "#9273E8", "#EAA852", "#D542E4", "#58A770", "#71EA4F", "#B395E5", "#5A7F8D", "#E6AB96"))+
    
    theme(legend.box.background = element_rect(color="white", size=2), panel.border = element_rect(colour = "black", fill=NA, size=2))+
      theme(axis.text = element_text(size = 12))+
     guides(fill=guide_legend(ncol = 1, title="B. Phylum|Genus"))+
     ggtitle("A. Absolute Abundance of Lake Cadagno Bacterial Communites (Genus)" )+
   theme(axis.text=element_text(size=14),
          axis.title=element_text(size=14,face="bold"), legend.text = element_text(size=12))+
        xlab("Depth (m)")+
    theme(
  # Hide panel borders and remove grid lines
  panel.grid.major = element_blank(),
  panel.grid.minor = element_blank(),
  # Change axis line 91A84C
  ) +
  #theme(axis.title.y = element_blank(), axis.ticks.y = element_blank())+
    theme(plot.title = element_text(size = 16, face = "bold"))+ 
   # theme(legend.key.width=unit(1.5, "cm"))+

  theme(legend.title=element_text(size=12, face="bold"), 
    legend.text=element_text(size=12), axis.ticks.y = element_blank())+
    theme(legend.position = "right")+
    coord_flip()+  scale_x_discrete(labels=c("LC-19m-E1"="19","LC-17m-E1"="17",
                              "LC-15point5m-E1"="15.5", 
                              "Cadagno15m-E2"="15", "LC-13m-E1"="13", "LC-11m-E1"="11","LC-9m-E1"="9",
                              "Cadagno7m-E2"="7", "LC-5m-E1"="5", "LC-3m-E1"="3"), 
                      limits=c("", "LC-19m-E1","LC-17m-E1"," ","LC-15point5m-E1", "Cadagno15m-E2",
                           "LC-13m-E1"," ", "LC-11m-E1", "LC-9m-E1", "Cadagno7m-E2",
                              "LC-5m-E1", "LC-3m-E1", ""))+
    ylab("Absolute Abundance in Cells/ml (>5000 Cells/mL) ")+
     annotate(geom="text", x=8, y=2e+05, label="Chemocline",
              color="black", size=5.5)+
      annotate(geom="text", x=14, y=2e+05, label="Mixolimnion",
              color="black", size=5.5) + 
      annotate(geom="text", x=4, y=2e+05, label="Monimolimnion",
              color="black", size=5.5)+
           guides(fill=guide_legend(ncol = 1, title="B. Phylum|Genus"))

    #Fig.S5
    fig.S5<-  pieipe3_ABN_Genus
    fig.S5
        
        

#Fig.S6
    

 
  turb_chro<- ggplot(Meta_data_PLP_VLP_II, aes(y=Meta_data_PLP_VLP_II$CTD_28_29_Turb, x=Meta_data_PLP_VLP_II$OTU_RA.Proteobacteria.Chromatium*100)) +
    geom_point() +
  stat_smooth(method = "lm", color = "black", size=0.3)+
  stat_cor(method = "pearson", label.y = 10, label.x = 0.1)+ theme_bw()+
    xlab("Chromatium Abundance (%)")+
    ylab("Turbidity (FTU)")+
     geom_point(aes(color=as.factor(Depth_m)), size=4)+
     scale_color_manual("Depth (m)", values=c("#52854C", "#C3D7A4","darkseagreen","forestgreen", "darkolivegreen3", "mediumorchid1", "deeppink4", "deeppink1", "#CC79A7", "peachpuff2", "peachpuff4"))
  
   PLP_chro<- ggplot(Meta_data_PLP_VLP_II, aes(y=Meta_data_PLP_VLP_II$TCell_cnt_ml, x=Meta_data_PLP_VLP_II$OTU_RA.Proteobacteria.Chromatium*100)) +
    geom_point() +
  stat_smooth(method = "lm", color = "black", size=0.3)+
  stat_cor(method = "pearson", label.y = 750000, label.x = 0.1)+ theme_bw()+
    xlab("Chromatium Abundance (%)")+
    ylab("PLPs/mL")+
     geom_point(aes(color=as.factor(Depth_m)), size=4)+
     scale_color_manual("Depth (m)", values=c("#52854C", "#C3D7A4","darkseagreen","forestgreen", "darkolivegreen3", "mediumorchid1", "deeppink4", "deeppink1", "#CC79A7", "peachpuff2", "peachpuff4"))
   
  
  Lentilm_L<- ggplot(Meta_data_PLP_VLP_II, aes(x=Meta_data_PLP_VLP_II$OTU_RA.Bacteroidetes.Lentimicrobiaceae_ge*100, y=Meta_data_PLP_VLP_II$Bacterial_production)) +
    geom_point() +
  stat_smooth(method = "lm", color = "black", size=0.3)+
  stat_cor(method = "pearson", label.y = 2.5e-06, label.x = 2)+ theme_bw()+
    xlab("Abundance of Lentimicrobium (%)")+
    ylab("Secondary Production (ngC L-1 H-1)")+
     geom_point(aes(color=as.factor(Depth_m)), size=4)+
     scale_color_manual("Depth (m)", values=c("#52854C", "#C3D7A4","darkseagreen","forestgreen", "darkolivegreen3", "mediumorchid1", "deeppink3", "deeppink4", "peachpuff2", "peachpuff4"))
  
 
  turb_lenti<- ggplot(Meta_data_PLP_VLP_II, aes(y=Meta_data_PLP_VLP_II$CTD_28_29_Turb, x=Meta_data_PLP_VLP_II$OTU_RA.Bacteroidetes.Lentimicrobiaceae_ge*100)) +
    geom_point() +
  stat_smooth(method = "lm", color = "black", size=0.3)+
  stat_cor(method = "pearson", label.y = 10, label.x = 2)+ theme_bw()+
    xlab("Lentimicrobiaceae_ge Abundance (%)")+
    ylab("Turbidity (FTU)")+
     geom_point(aes(color=as.factor(Depth_m)), size=4)+
     scale_color_manual("Depth (m)", values=c("#52854C", "#C3D7A4","darkseagreen","forestgreen", "darkolivegreen3", "mediumorchid1", "deeppink4", "deeppink1", "#CC79A7", "peachpuff2", "peachpuff4"))
  
    PLP_lenti<- ggplot(Meta_data_PLP_VLP_II, aes(y=Meta_data_PLP_VLP_II$TCell_cnt_ml, x=Meta_data_PLP_VLP_II$OTU_RA.Bacteroidetes.Lentimicrobiaceae_ge*100)) +
    geom_point() +
  stat_smooth(method = "lm", color = "black", size=0.3)+
  stat_cor(method = "pearson", label.y = 750000, label.x = 0.25)+ theme_bw()+
    xlab("Lentimicrobiaceae_ge Abundance (%)")+
    ylab("PLPs/mL")+
     geom_point(aes(color=as.factor(Depth_m)), size=4)+
     scale_color_manual("Depth (m)", values=c("#52854C", "#C3D7A4","darkseagreen","forestgreen", "darkolivegreen3", "mediumorchid1", "deeppink4", "deeppink1", "#CC79A7", "peachpuff2", "peachpuff4"))
    

    
     Lentilm_PON<- ggplot(Meta_data_PLP_VLP_II, aes(x=Meta_data_PLP_VLP_II$OTU_RA.Bacteroidetes.Lentimicrobiaceae_ge*100, y=Meta_data_PLP_VLP_II$PON)) +
    geom_point() +
  stat_smooth(method = "lm", color = "black", size=0.3)+
  stat_cor(method = "pearson", label.y = 1.2, label.x = 2)+ theme_bw()+
    xlab("Lentimicrobiaceae_ge Abundance (%)")+
    ylab("PON (ppm)")+
     geom_point(aes(color=as.factor(Depth_m)), size=4)+
     scale_color_manual("Depth (m)", values=c("#52854C", "#C3D7A4","darkseagreen","forestgreen", "darkolivegreen3", "mediumorchid1", "deeppink3", "deeppink4", "peachpuff2", "peachpuff4"))
     
     
     
          Lentilm_POS<- ggplot(Meta_data_PLP_VLP_II, aes(x=Meta_data_PLP_VLP_II$OTU_RA.Bacteroidetes.Lentimicrobiaceae_ge*100, y=Meta_data_PLP_VLP_II$POS)) +
    geom_point() +
  stat_smooth(method = "lm", color = "black", size=0.3)+
  stat_cor(method = "pearson", label.y = 0.5, label.x = 2)+ theme_bw()+
    xlab("Lentimicrobiaceae_ge Abundance (%)")+
    ylab("POS (ppm)")+
     geom_point(aes(color=as.factor(Depth_m)), size=4)+
     scale_color_manual("Depth (m)", values=c("#52854C", "#C3D7A4","darkseagreen","forestgreen", "darkolivegreen3", "mediumorchid1", "deeppink3", "deeppink4", "peachpuff2", "peachpuff4"))
          
        
               Lentilm_ligh<- ggplot(Meta_data_PLP_VLP_II, aes(x=Meta_data_PLP_VLP_II$OTU_RA.Bacteroidetes.Lentimicrobiaceae_ge*100, y=Meta_data_PLP_VLP_II$CTD2_PAR_29)) +
    geom_point() +
  stat_smooth(method = "lm", color = "black", size=0.3)+
  stat_cor(method = "pearson", label.y = 120, label.x = 2)+ theme_bw()+
    xlab("Lentimicrobiaceae_ge Abundance (%)")+
    ylab("Day 2 PAR (w/m^2)")+
     geom_point(aes(color=as.factor(Depth_m)), size=4)+
     scale_color_manual("Depth (m)", values=c("#52854C", "#C3D7A4","darkseagreen","forestgreen", "darkolivegreen3", "mediumorchid1", "deeppink3", "deeppink4", "peachpuff2", "peachpuff4"))
               
               
                  Lentilm_ox<- ggplot(Meta_data_PLP_VLP_II, aes(x=Meta_data_PLP_VLP_II$OTU_RA.Bacteroidetes.Lentimicrobiaceae_ge*100, y=Meta_data_PLP_VLP_II$CTD_28_29_Oxygen)) +
    geom_point() +
  stat_smooth(method = "lm", color = "black", size=0.3)+
  stat_cor(method = "pearson", label.y = 10, label.x = 2)+ theme_bw()+
    xlab("Abundance of Lentimicrobium (%)")+
    ylab("Day 1 and Day 2 Average Oxygen (mg/L)")+
     geom_point(aes(color=as.factor(Depth_m)), size=4)+
     scale_color_manual("Depth (m)", values=c("#52854C", "#C3D7A4","darkseagreen","forestgreen", "darkolivegreen3", "mediumorchid1", "deeppink3", "deeppink4", "peachpuff2", "peachpuff4"))
                  
                  
     Fig.S6<-     ggarrange(turb_chro, PLP_chro, Lentilm_L, turb_lenti,  PLP_lenti, Lentilm_PON, Lentilm_POS, Lentilm_ligh, Lentilm_ox, nrow = 5, ncol = 2, common.legend = T, legend= "bottom", labels = c("(A)", "(B)", "(C)"," (D)", "(E)", "(F)", "(G)", "(H)", "(I)"), align="v") #15x15 pdf
    
    
    
    
    #Fig.S6B
    
  
  
     PLP_Ery<- ggplot(Meta_data_PLP_VLP_II, aes(y=Meta_data_PLP_VLP_II$TCell_cnt_ml, x=Meta_data_PLP_VLP_II$OTU_RA.Firmicutes.Erysipelotrichaceae_unclassified*100)) +
    geom_point() +
  stat_smooth(method = "lm", color = "black", size=0.3)+
  stat_cor(method = "pearson", label.y = 750000, label.x = 1)+ theme_bw()+
    xlab("Erysipelotrichaceae_ge Abundance (%)")+
    ylab("PLPs/mL")+
     geom_point(aes(color=as.factor(Depth_m)), size=4)+
     scale_color_manual("Depth (m)", values=c("#52854C", "#C3D7A4","darkseagreen","forestgreen", "darkolivegreen3", "mediumorchid1", "deeppink4", "deeppink1", "#CC79A7", "peachpuff2", "peachpuff4"))
 
  turb_Ery<- ggplot(Meta_data_PLP_VLP_II, aes(y=Meta_data_PLP_VLP_II$CTD_28_29_Turb, x=Meta_data_PLP_VLP_II$OTU_RA.Firmicutes.Erysipelotrichaceae_unclassified*100)) +
    geom_point() +
  stat_smooth(method = "lm", color = "black", size=0.3)+
  stat_cor(method = "pearson", label.y = 10, label.x = 1)+ theme_bw()+
    xlab("Erysipelotrichaceae_ge Abundance (%)")+
    ylab("Turbidity (FTU)")+
     geom_point(aes(color=as.factor(Depth_m)), size=4)+
     scale_color_manual("Depth (m)", values=c("#52854C", "#C3D7A4","darkseagreen","forestgreen", "darkolivegreen3", "mediumorchid1", "deeppink4", "deeppink1", "#CC79A7", "peachpuff2", "peachpuff4"))
  
  PLP_acla<- ggplot(Meta_data_PLP_VLP_II, aes(y=Meta_data_PLP_VLP_II$TCell_cnt_ml, x=Meta_data_PLP_VLP_II$OTU_RA.Actinobacteria.acI.A*100)) +
    geom_point() +
  stat_smooth(method = "lm", color = "black", size=0.3)+
  stat_cor(method = "pearson", label.y = 750000, label.x = 4)+ theme_bw()+
    xlab("acI.A Abundance (%)")+
    ylab("PLPs/mL")+
     geom_point(aes(color=as.factor(Depth_m)), size=4)+
     scale_color_manual("Depth (m)", values=c("#52854C", "#C3D7A4","darkseagreen","forestgreen", "darkolivegreen3", "mediumorchid1", "deeppink4", "deeppink1", "#CC79A7", "peachpuff2", "peachpuff4"))
 
  turb_acla<- ggplot(Meta_data_PLP_VLP_II, aes(y=Meta_data_PLP_VLP_II$CTD_28_29_Turb, x=Meta_data_PLP_VLP_II$OTU_RA.Actinobacteria.acI.A*100)) +
    geom_point() +
  stat_smooth(method = "lm", color = "black", size=0.3)+
  stat_cor(method = "pearson", label.y = 10, label.x = 4)+ theme_bw()+
    xlab("acI.A Abundance (%)")+
    ylab("Turbidity (FTU)")+
     geom_point(aes(color=as.factor(Depth_m)), size=4)+
     scale_color_manual("Depth (m)", values=c("#52854C", "#C3D7A4","darkseagreen","forestgreen", "darkolivegreen3", "mediumorchid1", "deeppink4", "deeppink1", "#CC79A7", "peachpuff2", "peachpuff4"))
  
  
  
    PLP_aclb<- ggplot(Meta_data_PLP_VLP_II, aes(y=Meta_data_PLP_VLP_II$TCell_cnt_ml, x=Meta_data_PLP_VLP_II$OTU_RA.Actinobacteria.acI.A*100)) +
    geom_point() +
  stat_smooth(method = "lm", color = "black", size=0.3)+
  stat_cor(method = "pearson", label.y = 750000, label.x = 4)+ theme_bw()+
    xlab("acI.B Abundance (%)")+
    ylab("PLPs/mL")+
     geom_point(aes(color=as.factor(Depth_m)), size=4)+
     scale_color_manual("Depth (m)", values=c("#52854C", "#C3D7A4","darkseagreen","forestgreen", "darkolivegreen3", "mediumorchid1", "deeppink4", "deeppink1", "#CC79A7", "peachpuff2", "peachpuff4"))
 
  turb_aclb<- ggplot(Meta_data_PLP_VLP_II, aes(y=Meta_data_PLP_VLP_II$CTD_28_29_Turb, x=Meta_data_PLP_VLP_II$OTU_RA.Actinobacteria.acI.A*100)) +
    geom_point() +
  stat_smooth(method = "lm", color = "black", size=0.3)+
  stat_cor(method = "pearson", label.y = 10, label.x = 4)+ theme_bw()+
    xlab("acI.B Abundance (%)")+
    ylab("Turbidity (FTU)")+
     geom_point(aes(color=as.factor(Depth_m)), size=4)+
     scale_color_manual("Depth (m)", values=c("#52854C", "#C3D7A4","darkseagreen","forestgreen", "darkolivegreen3", "mediumorchid1", "deeppink4", "deeppink1", "#CC79A7", "peachpuff2", "peachpuff4"))
  

    
     Lentilm_E<- ggplot(grand_phen_g_CTD_lm, aes(x=grand_phen_g_CTD_lm$`OTU_RA.Firmicutes.Erysipelotrichaceae_unclassified`*100, y=grand_phen_g_CTD_lm$Bacterial_production)) +
    geom_point() +
  stat_smooth(method = "lm", color = "black", size=0.3)+ 
  stat_cor(method = "pearson", label.y = 2.5e-06, label.x = 1)+ theme_bw()+
    xlab("Abundance of Unclassified Erysipelotrichaceae (%)")+
    ylab("Bacterial Production")+
     geom_point(aes(color=as.factor(Depth_m)), size=4)+
     scale_color_manual("Depth (m)", values=c("#52854C", "#C3D7A4","darkseagreen","forestgreen", "darkolivegreen3", "mediumorchid1", "deeppink3", "deeppink4", "peachpuff2", "peachpuff4"))+
  
         ylim(0, 2.5e-06)

     
      Lentilm_CL<- ggplot(grand_phen_g_CTD_lm, aes(x=grand_phen_g_CTD_lm$`OTU_RA.Bacteroidetes.Chlorobium`*100, y=grand_phen_g_CTD_lm$Bacterial_production)) +
    geom_point() +
  stat_smooth(method = "lm", color = "black", size=0.3)+ 
  stat_cor(method = "pearson", label.y = 2.5e-06, label.x = 1)+ theme_bw()+
    xlab("Abundance of Chlorobium (%)")+
    ylab("Bacterial Production")+
     geom_point(aes(color=as.factor(Depth_m)), size=4)+
     scale_color_manual("Depth (m)", values=c("#52854C", "#C3D7A4","darkseagreen","forestgreen", "darkolivegreen3", "mediumorchid1", "deeppink3", "deeppink4", "peachpuff2", "peachpuff4"))
      
      
      Lentilm_Ery<- ggplot(Meta_data_PLP_VLP_II, aes(x=Meta_data_PLP_VLP_II$OTU_RA.Firmicutes.Erysipelotrichaceae_unclassified*100, y=Meta_data_PLP_VLP_II$POS)) +
    geom_point() +
  stat_smooth(method = "lm", color = "black", size=0.3)+ 
  stat_cor(method = "pearson", label.y = 0.5, label.x = 1)+ theme_bw()+
    xlab("Erysipelotrichaceae_ge Abundance (%)")+
    ylab("POS (ppm)")+
     geom_point(aes(color=as.factor(Depth_m)), size=4)+
     scale_color_manual("Depth (m)", values=c("#52854C", "#C3D7A4","darkseagreen","forestgreen", "darkolivegreen3", "mediumorchid1", "deeppink3", "deeppink4", "peachpuff2", "peachpuff4"))
              
     
      
  Fig.S6b<-     ggarrange(Lentilm_L, Lentilm_T, Lentilm_PON, Lentilm_POS, Lentilm_ox, Lentilm_ligh, Lentilm_E, Lentilm_CL, Lentilm_Ery, nrow = 5, ncol = 2, common.legend = T, align = "hv", legend= "bottom", labels = c("(A)", "(B)", "(C)"," (D)", "(E)", "(F)", "(G)", "(H)"))   
  
  
##Fig. S7u




  
  
  
  
  
  
  
  
  
  #Fig.S7

PLPD_turb<- ggplot(Meta_data_PLP_VLP_II, aes(y=Meta_data_PLP_VLP_II$C_D1_Alpha_div_Cells, x=Meta_data_PLP_VLP_II$CTD_28_29_Turb)) +
    geom_point() +
  stat_smooth(method = "lm", color = "black", size=0.3)+
  stat_cor(method = "pearson", label.y = 10000, label.x = 2)+ theme_bw()+
    xlab("Turbidity (FTU)")+
    ylab("PLPs Phenotypic Diversity (D1)")+
     geom_point(aes(color=as.factor(Depth_m)), size=4)+
     scale_color_manual("Depth (m)", values=c("#52854C", "#C3D7A4","darkseagreen","forestgreen", "darkolivegreen3", "mediumorchid1", "deeppink4", "deeppink1", "#CC79A7", "peachpuff2", "peachpuff4"))

PLPD_BP<- ggplot(Meta_data_PLP_VLP_II, aes(y=Meta_data_PLP_VLP_II$C_D1_Alpha_div_Cells, x=Meta_data_PLP_VLP_II$Bacterial_production)) +
    geom_point() +
  stat_smooth(method = "lm", color = "black", size=0.3)+
  stat_cor(method = "pearson", label.y = 10000, label.x = 0.0000005)+ theme_bw()+
    xlab("Secondary Production (ngC L-1 H-1)")+
    ylab("PLPs Phenotypic Diversity (D1)")+
     geom_point(aes(color=as.factor(Depth_m)), size=4)+
     scale_color_manual("Depth (m)", values=c("#52854C", "#C3D7A4","darkseagreen","forestgreen", "darkolivegreen3", "mediumorchid1", "deeppink4", "deeppink1", "#CC79A7", "peachpuff2", "peachpuff4"))


PLPD_chl<- ggplot(Meta_data_PLP_VLP_II, aes(y=Meta_data_PLP_VLP_II$C_D1_Alpha_div_Cells, x=Meta_data_PLP_VLP_II$CTD_28_29_chl)) +
    geom_point() +
  stat_smooth(method = "lm", color = "black", size=0.3)+
  stat_cor(method = "pearson", label.y = 10000, label.x = 5)+ theme_bw()+
    xlab("Chl (μg/L)")+
    ylab("PLPs Phenotypic Diversity (D1)")+
     geom_point(aes(color=as.factor(Depth_m)), size=4)+
     scale_color_manual("Depth (m)", values=c("#52854C", "#C3D7A4","darkseagreen","forestgreen", "darkolivegreen3", "mediumorchid1", "deeppink4", "deeppink1", "#CC79A7", "peachpuff2", "peachpuff4"))




PLPD_ox<- ggplot(Meta_data_PLP_VLP_II, aes(y=Meta_data_PLP_VLP_II$C_D1_Alpha_div_Cells, x=Meta_data_PLP_VLP_II$CTD_28_29_Oxygen)) +
    geom_point() +
  stat_smooth(method = "lm", color = "black", size=0.3)+
  stat_cor(method = "pearson", label.y = 10000, label.x = 5)+ theme_bw()+
    xlab("Day 1 and Day 2 Average Oxygen (mg/L)")+
    ylab("PLPs Phenotypic Diversity (D1)")+
     geom_point(aes(color=as.factor(Depth_m)), size=4)+
     scale_color_manual("Depth (m)", values=c("#52854C", "#C3D7A4","darkseagreen","forestgreen", "darkolivegreen3", "mediumorchid1", "deeppink4", "deeppink1", "#CC79A7", "peachpuff2", "peachpuff4"))


PLPD_PAR<- ggplot(Meta_data_PLP_VLP_II, aes(y=Meta_data_PLP_VLP_II$C_D1_Alpha_div_Cells, x=Meta_data_PLP_VLP_II$CTD2_PAR_29)) +
    geom_point() +
  stat_smooth(method = "lm", color = "black", size=0.3)+
  stat_cor(method = "pearson", label.y = 10000, label.x =100)+ theme_bw()+
    xlab("Day 2 PAR (w/m^2)")+
    ylab("PLPs Phenotypic Diversity (D1)")+
     geom_point(aes(color=as.factor(Depth_m)), size=4)+
     scale_color_manual("Depth (m)", values=c("#52854C", "#C3D7A4","darkseagreen","forestgreen", "darkolivegreen3", "mediumorchid1", "deeppink4", "deeppink1", "#CC79A7", "peachpuff2", "peachpuff4"))


B_POC<- ggplot(Meta_data_PLP_VLP_II, aes(y=Meta_data_PLP_VLP_II$Shannon_16s_a, x=Meta_data_PLP_VLP_II$POC)) +
    geom_point() +
  stat_smooth(method = "lm", color = "black", size=0.3)+
  stat_cor(method = "pearson", label.y = 3, label.x =2)+ theme_bw()+
    xlab("POC(ppm)")+
    ylab("Shannon Genotypic Diversity")+
     geom_point(aes(color=as.factor(Depth_m)), size=4)+
     scale_color_manual("Depth (m)", values=c("#52854C", "#C3D7A4","darkseagreen","forestgreen", "darkolivegreen3", "mediumorchid1", "deeppink4", "deeppink1", "#CC79A7", "peachpuff2", "peachpuff4"))


B_Fis<- ggplot(Meta_data_PLP_VLP_II, aes(y=Meta_data_PLP_VLP_II$Fisher_16s_a, x=Meta_data_PLP_VLP_II$CTD_28_29_Oxygen)) +
    geom_point() +
  stat_smooth(method = "lm", color = "black", size=0.3)+
  stat_cor(method = "pearson", label.y = 200, label.x =5)+ theme_bw()+
    xlab("Day 1 and Day 2 Average Oxygen (mg/L)")+
    ylab("Fisher Genotypic Diversity")+
     geom_point(aes(color=as.factor(Depth_m)), size=4)+
     scale_color_manual("Depth (m)", values=c("#52854C", "#C3D7A4","darkseagreen","forestgreen", "darkolivegreen3", "mediumorchid1", "deeppink4", "deeppink1", "#CC79A7", "peachpuff2", "peachpuff4"))

B_FisL<- ggplot(Meta_data_PLP_VLP_II, aes(y=Meta_data_PLP_VLP_II$Fisher_16s_a, x=Meta_data_PLP_VLP_II$Ammonium_ug_L)) +
    geom_point() +
  stat_smooth(method = "lm", color = "black", size=0.3)+
  stat_cor(method = "pearson", label.y = 200, label.x =5)+ theme_bw()+
    xlab("Ammonium (μg/L)")+
    ylab("Fisher Genotypic Diversity")+
     geom_point(aes(color=as.factor(Depth_m)), size=4)+
     scale_color_manual("Depth (m)", values=c("#52854C", "#C3D7A4","darkseagreen","forestgreen", "darkolivegreen3", "mediumorchid1", "deeppink4", "deeppink1", "#CC79A7", "peachpuff2", "peachpuff4"))


#Fig.S6c

 Fig.S6c<-     ggarrange(PLPD_turb, PLPD_BP, PLPD_chl, PLPD_ox, PLPD_PAR, nrow = 3, ncol = 2, common.legend = T, legend= "bottom", labels = c("(A)", "(B)", "(C)"," (D)", "(E)", "(F)"))


#Metadata
 Meta_data_PLP_VLP_II <-  read.csv("~/GitHub/Lake_Cadagno_microbial_loop_Saini_et_al_2021/Fig.1-4_Suppl/RA_div_all3.csv") 
 
 
 
 




#Fig.S5


  
  
  
  





        
        
         


#fig.S7
 
 PLPD_ox<- ggplot(Meta_data_PLP_VLP_II, aes(y=Meta_data_PLP_VLP_II$C_D1_Alpha_div_Cells, x=Meta_data_PLP_VLP_II$CTD_28_29_Oxygen)) +
    geom_point() +
  stat_smooth(method = "lm", color = "black", size=0.3)+
  stat_cor(method = "pearson", label.y = 10000, label.x = 5)+ theme_bw()+
    xlab("Day 1 and Day 2 Average Oxygen (mg/L)")+
    ylab("PLPs Phenotypic Diversity (D1)")+
     geom_point(aes(color=as.factor(Depth_m)), size=4)+
     scale_color_manual("Depth (m)", values=c("#52854C", "#C3D7A4","darkseagreen","forestgreen", "darkolivegreen3", "mediumorchid1", "deeppink4", "deeppink1", "#CC79A7", "peachpuff2", "peachpuff4"))+
   labs(title = "A. Oxygen vs PLPs Shannon alpha diversity")


PLPD_PAR<- ggplot(Meta_data_PLP_VLP_II, aes(y=Meta_data_PLP_VLP_II$C_D1_Alpha_div_Cells, x=Meta_data_PLP_VLP_II$CTD2_PAR_29)) +
    geom_point() +
  stat_smooth(method = "lm", color = "black", size=0.3)+
  stat_cor(method = "pearson", label.y = 10000, label.x =100)+ theme_bw()+
    xlab("Day 2 PAR (w/m^2)")+
    ylab("PLPs Shannon Phenotypic Diversity (D1)")+
     geom_point(aes(color=as.factor(Depth_m)), size=4)+
     scale_color_manual("Depth (m)", values=c("#52854C", "#C3D7A4","darkseagreen","forestgreen", "darkolivegreen3", "mediumorchid1", "deeppink4", "deeppink1", "#CC79A7", "peachpuff2", "peachpuff4"))+
  labs(title = "B. Light vs PLPs Shannon alpha diversity")



oxygen_pcoa<- ggplot(Meta_data_PLP_VLP_II, aes(CTD_28_29_Oxygen, Axis.1_16s_b)) +
  geom_point() +
  stat_smooth(color = "black", size=0.3, method = 'lm') +
  stat_cor(method = "pearson", label.y = 0.5, label.x = 1)+ theme_bw()+
     labs(title = "C. Oxygen vs 16S beta diversity (incl. Chloroplast)")+
    xlab("Day 1 and Day 2 Oxygen (mg/L)")+
        ylab("PCoA Axis.1 Variation (16s rRNA)")+
   # theme(axis.text=element_text(size=24),axis.title=element_text(size=24,face="bold"))+
  
  geom_point(aes(color=as.factor(Depth_m)), size=4)+
     scale_color_manual("Depth (m)", values=c("#52854C", "#C3D7A4","darkseagreen","forestgreen", "darkolivegreen3", "mediumorchid1", "deeppink4", "deeppink1", "#CC79A7", "peachpuff2", "peachpuff4"))


     light_pcoa<- ggplot(Meta_data_PLP_VLP_II, aes(CTD2_PAR_29, Axis.1_16s_b)) +
   geom_point() +
  stat_smooth(color = "black", size=0.3, method="lm") +
  stat_cor(method = "pearson", label.y = 0.5, label.x = 3)+ theme_bw() +
        labs(title = "D. Light vs 16S beta diversity (incl. Chloroplast)")+
    xlab("Day 2 PAR (w/m^2)")+
        ylab("PCoA Axis.1 Variation (16s rRNA)")+
      # theme(axis.text=element_text(size=24),axis.title=element_text(size=24,face="bold"))+
     #
geom_point(aes(color=as.factor(Depth_m)), size=4)+
     scale_color_manual("Depth (m)", values=c("#52854C", "#C3D7A4","darkseagreen","forestgreen", "darkolivegreen3", "mediumorchid1", "deeppink4", "deeppink1", "#CC79A7", "peachpuff2", "peachpuff4"))

    
  
       oxygen_pcoa2bv<- ggplot(Meta_data_PLP_VLP_II, aes(CTD_28_29_Oxygen, PCoA_Cells_SYBR_Axis1) ) +
  geom_point() +
  stat_smooth(color = "black", size=0.3, method="lm") +
  stat_cor(method = "pearson", label.y = 0.5, label.x = 1)+ theme_bw()+
     labs(title = "E. Oxygen vs PLPs phenotypic beta diversity")+
    xlab("Day 1 and Day 2 Oxygen (mg/l)")+
        ylab("PCoA Axis.1 Variation (Flow-Cytometry)")+
    #theme(axis.text=element_text(size=24),axis.title=element_text(size=24,face="bold"))+
  
  geom_point(aes(color=as.factor(Depth_m)), size=4)+
     scale_color_manual("Depth (m)", values=c("#52854C", "#C3D7A4","darkseagreen","forestgreen", "darkolivegreen3", "mediumorchid1", "deeppink4", "deeppink1", "#CC79A7", "peachpuff2", "peachpuff4"))
       
       
        
light_pcoa2<- ggplot(Meta_data_PLP_VLP_II, aes(CTD2_PAR_29, PCoA_Cells_SYBR_Axis1)) +
   geom_point() +
  stat_smooth(color = "black", size=0.3, method="lm") +
  stat_cor(method = "pearson", label.y = 0.5, label.x = 3)+ theme_bw() +
        labs(title = "F. Light vs PLPs phenotypic beta diversity")+
    xlab("Day 2 PAR (w/m^2)")+
        ylab("PCoA Axis.1 Variation (Flow-Cytometry)")+
      # theme(axis.text=element_text(size=24),axis.title=element_text(size=24,face="bold"))+
     #
geom_point(aes(color=as.factor(Depth_m)), size=4)+
     scale_color_manual("Depth (m)", values=c("#52854C", "#C3D7A4","darkseagreen","forestgreen", "darkolivegreen3", "mediumorchid1", "deeppink4", "deeppink1", "#CC79A7", "peachpuff2", "peachpuff4"))
          
       



  geno_p2<- ggplot(Meta_data_PLP_VLP_II, aes(y=C_D1_Alpha_div_Cells, x=Shannon_16s_a)) +
   geom_point() +
  stat_smooth(color = "black", size=0.3, method="lm") +
  stat_cor(method = "pearson", label.y = 10000, label.x = 3.5)+ theme_bw()+
        scale_color_manual("Depth (m)", values=c("#52854C", "#C3D7A4","darkseagreen","forestgreen", "darkolivegreen3", "mediumorchid1", "deeppink4", "deeppink1", "#CC79A7", "peachpuff2", "peachpuff4"))+
             geom_point(aes(color=as.factor(Depth_m)), size=4)+
       labs(title = "G. Phenotypic vs genotypic prokaryotic alpha diversity", subtitle = "Linear Regression, Method=LM")+
       xlab("Shannon Diversity (16s rRNA)")+
        ylab("Shannon Diversity, D1 (Flow-cytometry)")


     geno_p<- ggplot(Meta_data_PLP_VLP_II, aes(x=Meta_data_PLP_VLP_II$PCoA_Cells_SYBR_Axis1, y=Meta_data_PLP_VLP_II$Axis.1_16s_b)) +
   geom_point() +
  stat_smooth(color = "black", size=0.3, method = 'lm') +
  stat_cor(method = "pearson", label.y = -0.4, label.x = -0.2)+ theme_bw() +
       labs(title = "H. Phenotypic vs genotypic prokaryotic beta diversity")+
       xlab("PCoA Axis.1 Variation (16s rRNA)")+
        ylab("PCoA Axis.1 Variation (Flow-cytometry)")+
                    geom_point(aes(color=as.factor(Depth_m)), size=4)+
       scale_color_manual("Depth (m)", values=c("#52854C", "#C3D7A4","darkseagreen","forestgreen", "darkolivegreen3", "mediumorchid1", "deeppink4", "deeppink1", "#CC79A7", "peachpuff2", "peachpuff4"))

   
       
        
      Fig.S7<-     ggarrange(PLPD_ox, PLPD_PAR, oxygen_pcoa, light_pcoa, oxygen_pcoa2bv, light_pcoa2, geno_p2, geno_p, nrow = 4, ncol = 2, common.legend = T, legend= "bottom")
      Fig.S7
 
   


           
     
      ###Additional correlation plot
        

#Final Corrplot PLP
   
 Meta_data_PLPII <- read.csv("~/GitHub/Lake_Cadagno_microbial_loop_Saini_et_al_2021/Fig.1-4_Suppl/RA_div_all3.csv")
 
 
 PLP_cor<-Meta_data_PLPII[,-c(1)]
 

 #SI Table1

  PLP_cor2 <- PLP_cor[!names(PLP_cor) %in% c("TCell_cnt_SD", "F_mg_L", "Cl_mg_L", "Cyn_Cells_sd", "V_sd.D1", "V_sd.D2", "sd.D1_cells","sd.D2_cells", "C_Pheno_alpha_sd_D2", "se.chao1_16s_a", "se.ACE_16s_a", "uM_Chla.h", "pH_CTD1_29", "pH_CTD1_28", "avg_cyano_cnt", "PPng_CLh", "VLP.mL_SD", "Sample_name.y", "DO_mg_CTD1_29_cor", "Cell_specific_BP", "Organisation_cells", "DO_mg_l_CTD1_28",  "Turb_CTD1_29", "Turb_FTU_CTD1_28", "Conductivity_CTD2_29", "Temperature_CTD2_29", "InvSimpson_16s_a", "ACE_16s_a", "CV_cells", "Chl_A_CTD1_29", "Chl_A_ug_p_l_CTD1_28", "Temp_CTD1_28", "Temp_CTD1_29", "Cond_CTD1_29", "Cond_ms_p_cm_CTD1_28", "DO_mg_CTD1_29", "Sample_ID.x", "Sample_ID.y", "VLP_sd_ml", "VLP_p_ml")]

  PLP_cor3<- na.aggregate(PLP_cor2)
 
 
 # Function to get correletions and p.values in a "long" data frame
corr.data = function(PLP_cor3) {
  
  # Get correlations
  cor.vals = cor(PLP_cor3)
  
  # Get p-values
  cor.p = cor.mtest(PLP_cor3, conf.level = 0.95)$p
  rownames(cor.p) = rownames(cor.vals)
  colnames(cor.p) = colnames(cor.vals)
  
  cbind(rowvars=rownames(cor.vals), data.frame(cor.vals)) %>% 
    gather(colvars, corr, -rowvars) %>% 
    left_join(cbind(rowvars=rownames(cor.p), data.frame(cor.p)) %>% 
                gather(colvars, p.value, -rowvars))
}


corr.data(PLP_cor3) %>% 
  ggplot(aes(colvars, fct_rev(rowvars))) +
  geom_tile(colour="grey20", aes(fill=corr), size=0.5) +
  #geom_point(aes(size=p.value, colour=cut(abs(corr), c(0, 0.01, 0.05, 1), include.lowest=TRUE)), pch=15) +
  geom_text(aes(label=sprintf("%1.2f", corr)), position=position_nudge(y=0.2), 
            size=3, colour="white") +
  geom_text(aes(label=paste0("(",formatC(format = "e", p.value, digits=2),")")), position=position_nudge(y=-0.2), 
            size=1.75, colour="white") +
  scale_fill_gradient2("Pearson Correlation", low="red", mid="yellow2", high="blue", midpoint=0, limits=c(-1,1)) +
  scale_size_continuous(range=c(12,16)) +
  labs(x="",y="") +
  theme_classic() +
  coord_fixed()+
  #theme(axis.text.x = element_text(angle = 45, hjust = 1))+
   theme(panel.grid.major = element_blank(),
        panel.grid.minor = element_blank(),
        panel.border = element_rect(fill=NA,color="gray90", size=0.5, linetype="solid"),
        axis.line = element_blank(),
        axis.ticks = element_blank(),
        panel.background = element_rect(fill="gray90"),
        plot.background = element_rect(fill="gray90"),
        axis.text = element_text(color="black") )+
  scale_x_discrete(position = "top") +
    theme(axis.text.x = element_text(angle =-45, hjust=1))

        
        







 ##Figure 5

         fig5a<- ggarrange(Rich_sha, NULL, p1p2, NULL, widths = c(1.15,0.25,1,1), ncol = 4)
  
  fig5b<- betadiversity2  |  plot_chemoaAXXX2 + plot_layout(guides = "collect")
  
  fig5a / fig5b
       
       
       

    
       




```

